2022.03.26 12:28:15 INFO  tracing is disabled for protocol LSP, to enable tracing of incoming and outgoing JSON messages create an empty file at /Users/Tarin/Documents/Projects/Programming/spark_testing/.metals/lsp.trace.json or /Users/Tarin/Library/Caches/org.scalameta.metals/lsp.trace.json
2022.03.26 12:28:17 INFO  logging to file /Users/Tarin/Documents/Projects/Programming/spark_testing/.metals/metals.log
2022.03.26 12:28:17 INFO  Started: Metals version 0.11.2 in workspace '/Users/Tarin/Documents/Projects/Programming/spark_testing' for client Visual Studio Code 1.65.2.
2022.03.26 12:28:17 INFO  Parse release: 1.8.0_301
2022.03.26 12:28:17 WARN  Can't instantiate JavaInteractiveSemanticdb (version: Some(JdkVersion(1,8)), jdkHome: /Library/Internet Plug-Ins/JavaAppletPlugin.plugin/Contents/Home, javac exists: false)
2022.03.26 12:28:18 INFO  time: initialize in 1.31s
2022.03.26 12:28:20 WARN  Build server is not auto-connectable.
2022.03.26 12:28:20 WARN  no build tool detected in workspace '/Users/Tarin/Documents/Projects/Programming/spark_testing'. The most common cause for this problem is that the editor was opened in the wrong working directory, for example if you use sbt then the workspace directory should contain build.sbt. 
2022.03.26 12:28:21 WARN  no build target for: /Users/Tarin/Documents/Projects/Programming/spark_testing/src/main/scala/SimpleApp.scala
Mar 26, 2022 12:28:24 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
SEVERE: Internal error: java.lang.reflect.InvocationTargetException
java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
	at org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.lambda$null$0(GenericEndpoint.java:67)
	at org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.request(GenericEndpoint.java:120)
	at org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.handleRequest(RemoteEndpoint.java:261)
	at org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.consume(RemoteEndpoint.java:190)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.handleMessage(StreamMessageProducer.java:194)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.listen(StreamMessageProducer.java:94)
	at org.eclipse.lsp4j.jsonrpc.json.ConcurrentMessageProcessor.run(ConcurrentMessageProcessor.java:113)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.lambda$null$0(GenericEndpoint.java:65)
	... 11 more
Caused by: java.lang.IllegalArgumentException: 15 is not a valid line number, allowed [0..0]
	at scala.meta.internal.inputs.InternalInput.lineToOffset(InternalInput.scala:37)
	at scala.meta.internal.inputs.InternalInput.lineToOffset$(InternalInput.scala:32)
	at scala.meta.inputs.Input$VirtualFile.lineToOffset(Input.scala:80)
	at scala.meta.inputs.Position$Range$.apply(Position.scala:59)
	at scala.meta.internal.mtags.MtagsEnrichments$XtensionPositionLspInverse.toMeta(MtagsEnrichments.scala:143)
	at scala.meta.internal.parsing.Trees.$anonfun$findLastEnclosingAt$4(Trees.scala:78)
	at scala.Option.flatMap(Option.scala:271)
	at scala.meta.internal.parsing.Trees.findLastEnclosingAt(Trees.scala:77)
	at scala.meta.internal.metals.codeactions.PatternMatchRefactor.contribute(PatternMatchRefactor.scala:96)
	at scala.meta.internal.metals.CodeActionProvider$$anonfun$1.applyOrElse(CodeActionProvider.scala:64)
	at scala.meta.internal.metals.CodeActionProvider$$anonfun$1.applyOrElse(CodeActionProvider.scala:63)
	at scala.collection.immutable.List.collect(List.scala:323)
	at scala.meta.internal.metals.CodeActionProvider.codeActions(CodeActionProvider.scala:63)
	at scala.meta.internal.metals.MetalsLanguageServer.$anonfun$codeAction$1(MetalsLanguageServer.scala:1615)
	at scala.meta.internal.metals.CancelTokens$.future(CancelTokens.scala:38)
	at scala.meta.internal.metals.MetalsLanguageServer.codeAction(MetalsLanguageServer.scala:1614)
	... 16 more

2022.03.26 12:28:28 INFO  no build target found for /Users/Tarin/Documents/Projects/Programming/spark_testing/src/main/scala/SimpleApp.scala. Using presentation compiler with project's scala-library version: 3.1.1
2022.03.26 12:28:40 INFO  time: code lens generation in 19s
2022.03.26 12:28:40 INFO  time: code lens generation in 16s
2022.03.26 12:28:40 INFO  time: code lens generation in 17s
Mar 26, 2022 12:28:41 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
WARNING: Unmatched cancel notification for request id 8
2022.03.26 12:28:41 WARN  no build target for: /Users/Tarin/Documents/Projects/Programming/spark_testing/src/main/scala/SimpleApp.scala
Mar 26, 2022 12:28:41 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
SEVERE: Internal error: java.lang.reflect.InvocationTargetException
java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
	at org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.lambda$null$0(GenericEndpoint.java:67)
	at org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.request(GenericEndpoint.java:120)
	at org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.handleRequest(RemoteEndpoint.java:261)
	at org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.consume(RemoteEndpoint.java:190)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.handleMessage(StreamMessageProducer.java:194)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.listen(StreamMessageProducer.java:94)
	at org.eclipse.lsp4j.jsonrpc.json.ConcurrentMessageProcessor.run(ConcurrentMessageProcessor.java:113)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.lambda$null$0(GenericEndpoint.java:65)
	... 11 more
Caused by: java.lang.IllegalArgumentException: 15 is not a valid line number, allowed [0..0]
	at scala.meta.internal.inputs.InternalInput.lineToOffset(InternalInput.scala:37)
	at scala.meta.internal.inputs.InternalInput.lineToOffset$(InternalInput.scala:32)
	at scala.meta.inputs.Input$VirtualFile.lineToOffset(Input.scala:80)
	at scala.meta.inputs.Position$Range$.apply(Position.scala:59)
	at scala.meta.internal.mtags.MtagsEnrichments$XtensionPositionLspInverse.toMeta(MtagsEnrichments.scala:143)
	at scala.meta.internal.parsing.Trees.$anonfun$findLastEnclosingAt$4(Trees.scala:78)
	at scala.Option.flatMap(Option.scala:271)
	at scala.meta.internal.parsing.Trees.findLastEnclosingAt(Trees.scala:77)
	at scala.meta.internal.metals.codeactions.PatternMatchRefactor.contribute(PatternMatchRefactor.scala:96)
	at scala.meta.internal.metals.CodeActionProvider$$anonfun$1.applyOrElse(CodeActionProvider.scala:64)
	at scala.meta.internal.metals.CodeActionProvider$$anonfun$1.applyOrElse(CodeActionProvider.scala:63)
	at scala.collection.immutable.List.collect(List.scala:323)
	at scala.meta.internal.metals.CodeActionProvider.codeActions(CodeActionProvider.scala:63)
	at scala.meta.internal.metals.MetalsLanguageServer.$anonfun$codeAction$1(MetalsLanguageServer.scala:1615)
	at scala.meta.internal.metals.CancelTokens$.future(CancelTokens.scala:38)
	at scala.meta.internal.metals.MetalsLanguageServer.codeAction(MetalsLanguageServer.scala:1614)
	... 16 more

Mar 26, 2022 12:28:41 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
SEVERE: Internal error: java.lang.reflect.InvocationTargetException
java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
	at org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.lambda$null$0(GenericEndpoint.java:67)
	at org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.request(GenericEndpoint.java:120)
	at org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.handleRequest(RemoteEndpoint.java:261)
	at org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.consume(RemoteEndpoint.java:190)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.handleMessage(StreamMessageProducer.java:194)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.listen(StreamMessageProducer.java:94)
	at org.eclipse.lsp4j.jsonrpc.json.ConcurrentMessageProcessor.run(ConcurrentMessageProcessor.java:113)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.lambda$null$0(GenericEndpoint.java:65)
	... 11 more
Caused by: java.lang.IllegalArgumentException: 15 is not a valid line number, allowed [0..0]
	at scala.meta.internal.inputs.InternalInput.lineToOffset(InternalInput.scala:37)
	at scala.meta.internal.inputs.InternalInput.lineToOffset$(InternalInput.scala:32)
	at scala.meta.inputs.Input$VirtualFile.lineToOffset(Input.scala:80)
	at scala.meta.inputs.Position$Range$.apply(Position.scala:59)
	at scala.meta.internal.mtags.MtagsEnrichments$XtensionPositionLspInverse.toMeta(MtagsEnrichments.scala:143)
	at scala.meta.internal.parsing.Trees.$anonfun$findLastEnclosingAt$4(Trees.scala:78)
	at scala.Option.flatMap(Option.scala:271)
	at scala.meta.internal.parsing.Trees.findLastEnclosingAt(Trees.scala:77)
	at scala.meta.internal.metals.codeactions.PatternMatchRefactor.contribute(PatternMatchRefactor.scala:96)
	at scala.meta.internal.metals.CodeActionProvider$$anonfun$1.applyOrElse(CodeActionProvider.scala:64)
	at scala.meta.internal.metals.CodeActionProvider$$anonfun$1.applyOrElse(CodeActionProvider.scala:63)
	at scala.collection.immutable.List.collect(List.scala:323)
	at scala.meta.internal.metals.CodeActionProvider.codeActions(CodeActionProvider.scala:63)
	at scala.meta.internal.metals.MetalsLanguageServer.$anonfun$codeAction$1(MetalsLanguageServer.scala:1615)
	at scala.meta.internal.metals.CancelTokens$.future(CancelTokens.scala:38)
	at scala.meta.internal.metals.MetalsLanguageServer.codeAction(MetalsLanguageServer.scala:1614)
	... 16 more

2022.03.26 12:29:03 WARN  no build target for: /Users/Tarin/Documents/Projects/Programming/spark_testing/simple.sbt
2022.03.26 12:30:13 WARN  no build target for: /Users/Tarin/Documents/Projects/Programming/spark_testing/simple.sbt
2022.03.26 12:30:42 WARN  no build target for: /Users/Tarin/Documents/Projects/Programming/spark_testing/simple.sbt
2022.03.26 12:39:23 INFO  running '/Library/Internet Plug-Ins/JavaAppletPlugin.plugin/Contents/Home/bin/java -Djline.terminal=jline.UnsupportedTerminal -Dsbt.log.noformat=true -Dfile.encoding=UTF-8 -jar /var/folders/22/_cc1t3rx73j5wlxzbbxtj41c0000gn/T/metals1237863121810852942/sbt-launch.jar -Dbloop.export-jar-classifiers=sources bloopInstall'
2022.03.26 12:39:23 WARN  no build target for: /Users/Tarin/Documents/Projects/Programming/spark_testing/project/metals.sbt
2022.03.26 12:39:23 WARN  no build target for: /Users/Tarin/Documents/Projects/Programming/spark_testing/project/project/metals.sbt
2022.03.26 12:39:31 WARN  no build target for: /Users/Tarin/Documents/Projects/Programming/spark_testing/project/metals.sbt
2022.03.26 12:39:31 WARN  no build target for: /Users/Tarin/Documents/Projects/Programming/spark_testing/project/project/metals.sbt
2022.03.26 12:39:32 INFO  [info] welcome to sbt 1.6.2 (Oracle Corporation Java 1.8.0_301)
2022.03.26 12:39:48 INFO  [info] loading settings for project spark_testing-build-build from metals.sbt ...
2022.03.26 12:39:51 INFO  [info] loading project definition from /Users/Tarin/Documents/Projects/Programming/spark_testing/project/project
2022.03.26 12:40:16 INFO  [info] loading settings for project spark_testing-build from metals.sbt ...
2022.03.26 12:40:16 INFO  [info] loading project definition from /Users/Tarin/Documents/Projects/Programming/spark_testing/project
2022.03.26 12:40:20 INFO  skipping build import with status 'Started'
2022.03.26 12:40:38 INFO  [success] Generated .bloop/spark_testing-build.json
2022.03.26 12:40:38 INFO  [success] Total time: 22 s, completed 26/03/2022 12:40:39 PM
2022.03.26 12:40:49 INFO  [info] loading settings for project spark_testing from simple.sbt ...
2022.03.26 12:40:49 INFO  [info] set current project to Simple Project (in build file:/Users/Tarin/Documents/Projects/Programming/spark_testing/)
2022.03.26 12:41:17 INFO  [success] Generated .bloop/spark_testing-test.json
2022.03.26 12:41:17 INFO  [success] Generated .bloop/spark_testing.json
2022.03.26 12:41:17 INFO  [success] Total time: 27 s, completed 26/03/2022 12:41:17 PM
2022.03.26 12:41:17 INFO  time: ran 'sbt bloopInstall' in 1m54s
2022.03.26 12:41:17 INFO  Attempting to connect to the build server...
2022.03.26 12:41:52 INFO  tracing is disabled for protocol BSP, to enable tracing of incoming and outgoing JSON messages create an empty file at /Users/Tarin/Documents/Projects/Programming/spark_testing/.metals/bsp.trace.json or /Users/Tarin/Library/Caches/org.scalameta.metals/bsp.trace.json
2022.03.26 12:42:01 INFO  Attempting to connect to the build server...
2022.03.26 12:42:01 INFO  tracing is disabled for protocol BSP, to enable tracing of incoming and outgoing JSON messages create an empty file at /Users/Tarin/Documents/Projects/Programming/spark_testing/project/.metals/bsp.trace.json or /Users/Tarin/Library/Caches/org.scalameta.metals/bsp.trace.json
2022.03.26 12:42:06 INFO  time: Connected to build server in 48s
2022.03.26 12:42:06 INFO  Connected to Build server: Bloop v1.4.13
2022.03.26 12:42:06 INFO  time: Imported build in 0.27s
2022.03.26 12:42:18 WARN  Could not find java sources in /Library/Internet Plug-Ins/JavaAppletPlugin.plugin/Contents/src.zip, /Library/Internet Plug-Ins/JavaAppletPlugin.plugin/Contents/lib/src.zip, /Library/Internet Plug-Ins/JavaAppletPlugin.plugin/Contents/Home/src.zip, /Library/Internet Plug-Ins/JavaAppletPlugin.plugin/Contents/Home/lib/src.zip. Java symbols will not be available.
2022.03.26 12:42:31 INFO  time: indexed workspace in 24s
2022.03.26 12:42:33 INFO  compiling spark_testing (1 scala source)
2022.03.26 12:43:00 INFO  time: code lens generation in 27s
2022.03.26 12:43:00 INFO  time: code lens generation in 27s
2022.03.26 12:43:39 WARN  1 deprecation (since 2.13.0); re-run with -deprecation for details
2022.03.26 12:43:39 INFO  time: compiled spark_testing in 1m6s
Mar 26, 2022 12:48:38 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
SEVERE: Internal error: java.lang.reflect.InvocationTargetException
java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
	at org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.lambda$null$0(GenericEndpoint.java:67)
	at org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.request(GenericEndpoint.java:120)
	at org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.handleRequest(RemoteEndpoint.java:261)
	at org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.consume(RemoteEndpoint.java:190)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.handleMessage(StreamMessageProducer.java:194)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.listen(StreamMessageProducer.java:94)
	at org.eclipse.lsp4j.jsonrpc.json.ConcurrentMessageProcessor.run(ConcurrentMessageProcessor.java:113)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.lambda$null$0(GenericEndpoint.java:65)
	... 11 more
Caused by: java.lang.IllegalArgumentException: 15 is not a valid line number, allowed [0..0]
	at scala.meta.internal.inputs.InternalInput.lineToOffset(InternalInput.scala:37)
	at scala.meta.internal.inputs.InternalInput.lineToOffset$(InternalInput.scala:32)
	at scala.meta.inputs.Input$VirtualFile.lineToOffset(Input.scala:80)
	at scala.meta.inputs.Position$Range$.apply(Position.scala:59)
	at scala.meta.internal.mtags.MtagsEnrichments$XtensionPositionLspInverse.toMeta(MtagsEnrichments.scala:143)
	at scala.meta.internal.parsing.Trees.$anonfun$findLastEnclosingAt$4(Trees.scala:78)
	at scala.Option.flatMap(Option.scala:271)
	at scala.meta.internal.parsing.Trees.findLastEnclosingAt(Trees.scala:77)
	at scala.meta.internal.metals.codeactions.PatternMatchRefactor.contribute(PatternMatchRefactor.scala:96)
	at scala.meta.internal.metals.CodeActionProvider$$anonfun$1.applyOrElse(CodeActionProvider.scala:64)
	at scala.meta.internal.metals.CodeActionProvider$$anonfun$1.applyOrElse(CodeActionProvider.scala:63)
	at scala.collection.immutable.List.collect(List.scala:323)
	at scala.meta.internal.metals.CodeActionProvider.codeActions(CodeActionProvider.scala:63)
	at scala.meta.internal.metals.MetalsLanguageServer.$anonfun$codeAction$1(MetalsLanguageServer.scala:1615)
	at scala.meta.internal.metals.CancelTokens$.future(CancelTokens.scala:38)
	at scala.meta.internal.metals.MetalsLanguageServer.codeAction(MetalsLanguageServer.scala:1614)
	... 16 more

Mar 26, 2022 12:48:44 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
SEVERE: Internal error: java.lang.reflect.InvocationTargetException
java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
	at org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.lambda$null$0(GenericEndpoint.java:67)
	at org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.request(GenericEndpoint.java:120)
	at org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.handleRequest(RemoteEndpoint.java:261)
	at org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.consume(RemoteEndpoint.java:190)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.handleMessage(StreamMessageProducer.java:194)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.listen(StreamMessageProducer.java:94)
	at org.eclipse.lsp4j.jsonrpc.json.ConcurrentMessageProcessor.run(ConcurrentMessageProcessor.java:113)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.lambda$null$0(GenericEndpoint.java:65)
	... 11 more
Caused by: java.lang.IllegalArgumentException: 6 is not a valid line number, allowed [0..0]
	at scala.meta.internal.inputs.InternalInput.lineToOffset(InternalInput.scala:37)
	at scala.meta.internal.inputs.InternalInput.lineToOffset$(InternalInput.scala:32)
	at scala.meta.inputs.Input$VirtualFile.lineToOffset(Input.scala:80)
	at scala.meta.inputs.Position$Range$.apply(Position.scala:59)
	at scala.meta.internal.mtags.MtagsEnrichments$XtensionPositionLspInverse.toMeta(MtagsEnrichments.scala:143)
	at scala.meta.internal.parsing.Trees.$anonfun$findLastEnclosingAt$4(Trees.scala:78)
	at scala.Option.flatMap(Option.scala:271)
	at scala.meta.internal.parsing.Trees.findLastEnclosingAt(Trees.scala:77)
	at scala.meta.internal.metals.codeactions.PatternMatchRefactor.contribute(PatternMatchRefactor.scala:96)
	at scala.meta.internal.metals.CodeActionProvider$$anonfun$1.applyOrElse(CodeActionProvider.scala:64)
	at scala.meta.internal.metals.CodeActionProvider$$anonfun$1.applyOrElse(CodeActionProvider.scala:63)
	at scala.collection.immutable.List.collect(List.scala:323)
	at scala.meta.internal.metals.CodeActionProvider.codeActions(CodeActionProvider.scala:63)
	at scala.meta.internal.metals.MetalsLanguageServer.$anonfun$codeAction$1(MetalsLanguageServer.scala:1615)
	at scala.meta.internal.metals.CancelTokens$.future(CancelTokens.scala:38)
	at scala.meta.internal.metals.MetalsLanguageServer.codeAction(MetalsLanguageServer.scala:1614)
	... 16 more

Mar 26, 2022 12:48:45 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
SEVERE: Internal error: java.lang.reflect.InvocationTargetException
java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
	at org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.lambda$null$0(GenericEndpoint.java:67)
	at org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.request(GenericEndpoint.java:120)
	at org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.handleRequest(RemoteEndpoint.java:261)
	at org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.consume(RemoteEndpoint.java:190)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.handleMessage(StreamMessageProducer.java:194)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.listen(StreamMessageProducer.java:94)
	at org.eclipse.lsp4j.jsonrpc.json.ConcurrentMessageProcessor.run(ConcurrentMessageProcessor.java:113)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.lambda$null$0(GenericEndpoint.java:65)
	... 11 more
Caused by: java.lang.IllegalArgumentException: 7 is not a valid line number, allowed [0..0]
	at scala.meta.internal.inputs.InternalInput.lineToOffset(InternalInput.scala:37)
	at scala.meta.internal.inputs.InternalInput.lineToOffset$(InternalInput.scala:32)
	at scala.meta.inputs.Input$VirtualFile.lineToOffset(Input.scala:80)
	at scala.meta.inputs.Position$Range$.apply(Position.scala:59)
	at scala.meta.internal.mtags.MtagsEnrichments$XtensionPositionLspInverse.toMeta(MtagsEnrichments.scala:143)
	at scala.meta.internal.parsing.Trees.$anonfun$findLastEnclosingAt$4(Trees.scala:78)
	at scala.Option.flatMap(Option.scala:271)
	at scala.meta.internal.parsing.Trees.findLastEnclosingAt(Trees.scala:77)
	at scala.meta.internal.metals.codeactions.PatternMatchRefactor.contribute(PatternMatchRefactor.scala:96)
	at scala.meta.internal.metals.CodeActionProvider$$anonfun$1.applyOrElse(CodeActionProvider.scala:64)
	at scala.meta.internal.metals.CodeActionProvider$$anonfun$1.applyOrElse(CodeActionProvider.scala:63)
	at scala.collection.immutable.List.collect(List.scala:323)
	at scala.meta.internal.metals.CodeActionProvider.codeActions(CodeActionProvider.scala:63)
	at scala.meta.internal.metals.MetalsLanguageServer.$anonfun$codeAction$1(MetalsLanguageServer.scala:1615)
	at scala.meta.internal.metals.CancelTokens$.future(CancelTokens.scala:38)
	at scala.meta.internal.metals.MetalsLanguageServer.codeAction(MetalsLanguageServer.scala:1614)
	... 16 more

Mar 26, 2022 12:48:59 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
WARNING: Unmatched cancel notification for request id 79
Mar 26, 2022 12:48:59 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
WARNING: Unmatched cancel notification for request id 82
2022.03.26 12:49:01 INFO  compiling spark_testing (1 scala source)
2022.03.26 12:49:03 WARN  1 deprecation (since 2.13.0); re-run with -deprecation for details
2022.03.26 12:49:03 INFO  time: compiled spark_testing in 1.97s
2022.03.26 12:50:55 INFO  compiling spark_testing (1 scala source)
2022.03.26 12:50:57 WARN  1 deprecation (since 2.13.0); re-run with -deprecation for details
2022.03.26 12:50:57 INFO  time: compiled spark_testing in 2.32s
Mar 26, 2022 12:52:02 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleNotification
WARNING: Notification threw an exception: {
  "jsonrpc": "2.0",
  "method": "workspace/didChangeWatchedFiles",
  "params": {
    "changes": [
      {
        "uri": "file:///Users/Tarin/Documents/Projects/Programming/spark_testing/flightdata.sbt",
        "type": 1
      },
      {
        "uri": "file:///Users/Tarin/Documents/Projects/Programming/spark_testing/simple.sbt",
        "type": 3
      }
    ]
  }
}
java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
	at org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.lambda$null$0(GenericEndpoint.java:67)
	at org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.notify(GenericEndpoint.java:152)
	at org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.handleNotification(RemoteEndpoint.java:220)
	at org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.consume(RemoteEndpoint.java:187)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.handleMessage(StreamMessageProducer.java:194)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.listen(StreamMessageProducer.java:94)
	at org.eclipse.lsp4j.jsonrpc.json.ConcurrentMessageProcessor.run(ConcurrentMessageProcessor.java:113)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.lambda$null$0(GenericEndpoint.java:65)
	... 11 more
Caused by: java.nio.file.NoSuchFileException: /Users/Tarin/Documents/Projects/Programming/spark_testing/simple.sbt
	at sun.nio.fs.UnixException.translateToIOException(UnixException.java:86)
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:102)
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:107)
	at sun.nio.fs.UnixFileSystemProvider.newByteChannel(UnixFileSystemProvider.java:214)
	at java.nio.file.Files.newByteChannel(Files.java:361)
	at java.nio.file.Files.newByteChannel(Files.java:407)
	at java.nio.file.Files.readAllBytes(Files.java:3152)
	at scala.meta.internal.io.PlatformFileIO$.slurp(PlatformFileIO.scala:45)
	at scala.meta.internal.io.FileIO$.slurp(FileIO.scala:24)
	at scala.meta.internal.metals.MetalsLanguageServer.$anonfun$onChange$1(MetalsLanguageServer.scala:1389)
	at scala.meta.internal.metals.MetalsLanguageServer.$anonfun$onChange$1$adapted(MetalsLanguageServer.scala:1388)
	at scala.collection.immutable.Stream.foreach(Stream.scala:533)
	at scala.meta.internal.metals.MetalsLanguageServer.onChange(MetalsLanguageServer.scala:1388)
	at scala.meta.internal.metals.MetalsLanguageServer.didChangeWatchedFiles(MetalsLanguageServer.scala:1332)
	... 16 more

2022.03.26 12:52:07 INFO  compiling spark_testing (1 scala source)
Mar 26, 2022 12:52:07 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
WARNING: Unmatched cancel notification for request id 156
2022.03.26 12:52:09 ERROR searching for `_empty_/SimpleApp.main().(args)` failed
java.nio.file.NoSuchFileException: /Users/Tarin/Documents/Projects/Programming/spark_testing/src/main/scala/SimpleApp.scala
	at sun.nio.fs.UnixException.translateToIOException(UnixException.java:86)
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:102)
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:107)
	at sun.nio.fs.UnixFileSystemProvider.newByteChannel(UnixFileSystemProvider.java:214)
	at java.nio.file.Files.newByteChannel(Files.java:361)
	at java.nio.file.Files.newByteChannel(Files.java:407)
	at java.nio.file.Files.readAllBytes(Files.java:3152)
	at scala.meta.internal.io.PlatformFileIO$.slurp(PlatformFileIO.scala:45)
	at scala.meta.internal.io.FileIO$.slurp(FileIO.scala:24)
	at scala.meta.internal.mtags.CommonMtagsEnrichments$XtensionAbsolutePath.toInput(CommonMtagsEnrichments.scala:521)
	at scala.meta.internal.mtags.SymbolIndexBucket.addMtagsSourceFile(SymbolIndexBucket.scala:198)
	at scala.meta.internal.mtags.SymbolIndexBucket.$anonfun$query0$1(SymbolIndexBucket.scala:163)
	at scala.meta.internal.mtags.SymbolIndexBucket.$anonfun$query0$1$adapted(SymbolIndexBucket.scala:163)
	at scala.collection.immutable.Set$Set2.foreach(Set.scala:181)
	at scala.meta.internal.mtags.SymbolIndexBucket.query0(SymbolIndexBucket.scala:163)
	at scala.meta.internal.mtags.SymbolIndexBucket.query(SymbolIndexBucket.scala:140)
	at scala.meta.internal.mtags.OnDemandSymbolIndex.$anonfun$findSymbolDefinition$1(OnDemandSymbolIndex.scala:133)
	at scala.collection.immutable.List.flatMap(List.scala:366)
	at scala.meta.internal.mtags.OnDemandSymbolIndex.findSymbolDefinition(OnDemandSymbolIndex.scala:133)
	at scala.meta.internal.mtags.OnDemandSymbolIndex.definition(OnDemandSymbolIndex.scala:44)
	at scala.meta.internal.metals.Docstrings.indexSymbol(Docstrings.scala:71)
	at scala.meta.internal.metals.Docstrings.documentation(Docstrings.scala:39)
	at scala.meta.internal.metals.MetalsSymbolSearch.documentation(MetalsSymbolSearch.scala:38)
	at scala.meta.internal.pc.MetalsGlobal.symbolDocumentation(MetalsGlobal.scala:162)
	at scala.meta.internal.pc.Signatures$SignaturePrinter.<init>(Signatures.scala:277)
	at scala.meta.internal.pc.HoverProvider.toHover(HoverProvider.scala:237)
	at scala.meta.internal.pc.HoverProvider.hoverOffset(HoverProvider.scala:103)
	at scala.meta.internal.pc.HoverProvider.hover(HoverProvider.scala:22)
	at scala.meta.internal.pc.ScalaPresentationCompiler.$anonfun$hover$1(ScalaPresentationCompiler.scala:192)
	at scala.meta.internal.pc.CompilerAccess.withSharedCompiler(CompilerAccess.scala:137)
	at scala.meta.internal.pc.CompilerAccess.$anonfun$withNonInterruptableCompiler$1(CompilerAccess.scala:125)
	at scala.meta.internal.pc.CompilerAccess.$anonfun$onCompilerJobQueue$1(CompilerAccess.scala:197)
	at scala.meta.internal.pc.CompilerJobQueue$Job.run(CompilerJobQueue.scala:139)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)

2022.03.26 12:52:10 WARN  1 deprecation (since 2.13.0); re-run with -deprecation for details
2022.03.26 12:52:10 INFO  time: compiled spark_testing in 2.9s
2022.03.26 12:52:13 INFO  compiling spark_testing (1 scala source)
2022.03.26 12:52:15 WARN  1 deprecation (since 2.13.0); re-run with -deprecation for details
2022.03.26 12:52:15 INFO  time: compiled spark_testing in 1.52s
2022.03.26 13:02:46 ERROR text document: file:///Users/Tarin/Documents/Projects/Programming/spark_testing/flightdata.sbt
java.nio.file.NoSuchFileException: /Users/Tarin/Documents/Projects/Programming/spark_testing/flightdata.sbt
	at sun.nio.fs.UnixException.translateToIOException(UnixException.java:86)
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:102)
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:107)
	at sun.nio.fs.UnixFileSystemProvider.newByteChannel(UnixFileSystemProvider.java:214)
	at java.nio.file.Files.newByteChannel(Files.java:361)
	at java.nio.file.Files.newByteChannel(Files.java:407)
	at java.nio.file.Files.readAllBytes(Files.java:3152)
	at scala.meta.internal.io.PlatformFileIO$.slurp(PlatformFileIO.scala:45)
	at scala.meta.internal.io.FileIO$.slurp(FileIO.scala:24)
	at scala.meta.internal.metals.InteractiveSemanticdbs.$anonfun$textDocument$2(InteractiveSemanticdbs.scala:90)
	at scala.Option.getOrElse(Option.scala:189)
	at scala.meta.internal.metals.InteractiveSemanticdbs.$anonfun$textDocument$1(InteractiveSemanticdbs.scala:90)
	at java.util.HashMap.compute(HashMap.java:1197)
	at java.util.Collections$SynchronizedMap.compute(Collections.java:2684)
	at scala.meta.internal.metals.InteractiveSemanticdbs.textDocument(InteractiveSemanticdbs.scala:89)
	at scala.meta.internal.metals.InteractiveSemanticdbs.textDocument(InteractiveSemanticdbs.scala:63)
	at scala.meta.internal.metals.AggregateSemanticdbs.loop$1(AggregateSemanticdbs.scala:30)
	at scala.meta.internal.metals.AggregateSemanticdbs.textDocument(AggregateSemanticdbs.scala:36)
	at scala.meta.internal.decorations.SyntheticsDecorationProvider.currentDocument(SyntheticsDecorationProvider.scala:278)
	at scala.meta.internal.decorations.SyntheticsDecorationProvider.syntheticDecorations(SyntheticsDecorationProvider.scala:165)
	at scala.meta.internal.decorations.SyntheticsDecorationProvider.$anonfun$publishSynthetics$1(SyntheticsDecorationProvider.scala:65)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
	at scala.util.Success.$anonfun$map$1(Try.scala:255)
	at scala.util.Success.map(Try.scala:213)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)

2022.03.26 13:02:46 ERROR text document: file:///Users/Tarin/Documents/Projects/Programming/spark_testing/flightdata.sbt
java.nio.file.NoSuchFileException: /Users/Tarin/Documents/Projects/Programming/spark_testing/flightdata.sbt
	at sun.nio.fs.UnixException.translateToIOException(UnixException.java:86)
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:102)
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:107)
	at sun.nio.fs.UnixFileSystemProvider.newByteChannel(UnixFileSystemProvider.java:214)
	at java.nio.file.Files.newByteChannel(Files.java:361)
	at java.nio.file.Files.newByteChannel(Files.java:407)
	at java.nio.file.Files.readAllBytes(Files.java:3152)
	at scala.meta.internal.io.PlatformFileIO$.slurp(PlatformFileIO.scala:45)
	at scala.meta.internal.io.FileIO$.slurp(FileIO.scala:24)
	at scala.meta.internal.metals.InteractiveSemanticdbs.$anonfun$textDocument$2(InteractiveSemanticdbs.scala:90)
	at scala.Option.getOrElse(Option.scala:189)
	at scala.meta.internal.metals.InteractiveSemanticdbs.$anonfun$textDocument$1(InteractiveSemanticdbs.scala:90)
	at java.util.HashMap.compute(HashMap.java:1197)
	at java.util.Collections$SynchronizedMap.compute(Collections.java:2684)
	at scala.meta.internal.metals.InteractiveSemanticdbs.textDocument(InteractiveSemanticdbs.scala:89)
	at scala.meta.internal.metals.InteractiveSemanticdbs.textDocument(InteractiveSemanticdbs.scala:63)
	at scala.meta.internal.metals.AggregateSemanticdbs.loop$1(AggregateSemanticdbs.scala:30)
	at scala.meta.internal.metals.AggregateSemanticdbs.textDocument(AggregateSemanticdbs.scala:36)
	at scala.meta.internal.metals.CodeLensProvider.findLenses(CodeLensProvider.scala:22)
	at scala.meta.internal.metals.MetalsLanguageServer.$anonfun$codeLens$2(MetalsLanguageServer.scala:1628)
	at scala.meta.internal.metals.TimerProvider.timedThunk(TimerProvider.scala:25)
	at scala.meta.internal.metals.MetalsLanguageServer.$anonfun$codeLens$1(MetalsLanguageServer.scala:1626)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$apply$2(CancelTokens.scala:26)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
	at scala.util.Success.$anonfun$map$1(Try.scala:255)
	at scala.util.Success.map(Try.scala:213)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)

2022.03.26 13:02:56 INFO  shutting down Metals
2022.03.26 13:02:56 INFO  Shut down connection with build server.
2022.03.26 13:02:56 INFO  Shut down connection with build server.
2022.03.26 13:03:30 INFO  tracing is disabled for protocol LSP, to enable tracing of incoming and outgoing JSON messages create an empty file at /Users/Tarin/Documents/Projects/Programming/flight-data/.metals/lsp.trace.json or /Users/Tarin/Library/Caches/org.scalameta.metals/lsp.trace.json
2022.03.26 13:03:32 INFO  logging to file /Users/Tarin/Documents/Projects/Programming/flight-data/.metals/metals.log
2022.03.26 13:03:32 INFO  Started: Metals version 0.11.2 in workspace '/Users/Tarin/Documents/Projects/Programming/flight-data' for client Visual Studio Code 1.65.2.
2022.03.26 13:03:32 INFO  Parse release: 1.8.0_301
2022.03.26 13:03:32 WARN  Can't instantiate JavaInteractiveSemanticdb (version: Some(JdkVersion(1,8)), jdkHome: /Library/Internet Plug-Ins/JavaAppletPlugin.plugin/Contents/Home, javac exists: false)
2022.03.26 13:03:33 INFO  time: initialize in 1.38s
2022.03.26 13:03:39 INFO  Attempting to connect to the build server...
2022.03.26 13:03:40 INFO  tracing is disabled for protocol BSP, to enable tracing of incoming and outgoing JSON messages create an empty file at /Users/Tarin/Documents/Projects/Programming/flight-data/.metals/bsp.trace.json or /Users/Tarin/Library/Caches/org.scalameta.metals/bsp.trace.json
2022.03.26 13:03:40 INFO  Attempting to connect to the build server...
2022.03.26 13:03:40 INFO  tracing is disabled for protocol BSP, to enable tracing of incoming and outgoing JSON messages create an empty file at /Users/Tarin/Documents/Projects/Programming/flight-data/project/.metals/bsp.trace.json or /Users/Tarin/Library/Caches/org.scalameta.metals/bsp.trace.json
2022.03.26 13:03:40 INFO  time: Connected to build server in 1.07s
2022.03.26 13:03:40 INFO  Connected to Build server: Bloop v1.4.13
2022.03.26 13:03:40 INFO  time: Imported build in 0.15s
2022.03.26 13:03:53 WARN  Could not find java sources in /Library/Internet Plug-Ins/JavaAppletPlugin.plugin/Contents/src.zip, /Library/Internet Plug-Ins/JavaAppletPlugin.plugin/Contents/lib/src.zip, /Library/Internet Plug-Ins/JavaAppletPlugin.plugin/Contents/Home/src.zip, /Library/Internet Plug-Ins/JavaAppletPlugin.plugin/Contents/Home/lib/src.zip. Java symbols will not be available.
2022.03.26 13:03:57 INFO  time: indexed workspace in 17s
2022.03.26 13:04:46 WARN  no build target for: /Users/Tarin/Documents/Projects/Programming/flight-data/src/main/scala/FlightData.scala
Mar 26, 2022 1:04:47 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
SEVERE: Internal error: java.util.NoSuchElementException: BatchedFunction
java.util.concurrent.CompletionException: java.util.NoSuchElementException: BatchedFunction
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:292)
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:308)
	at java.util.concurrent.CompletableFuture.uniAccept(CompletableFuture.java:661)
	at java.util.concurrent.CompletableFuture.uniAcceptStage(CompletableFuture.java:683)
	at java.util.concurrent.CompletableFuture.thenAccept(CompletableFuture.java:2010)
	at org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.handleRequest(RemoteEndpoint.java:279)
	at org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.consume(RemoteEndpoint.java:190)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.handleMessage(StreamMessageProducer.java:194)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.listen(StreamMessageProducer.java:94)
	at org.eclipse.lsp4j.jsonrpc.json.ConcurrentMessageProcessor.run(ConcurrentMessageProcessor.java:113)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.util.NoSuchElementException: BatchedFunction
	at scala.meta.internal.metals.BatchedFunction.<init>(BatchedFunction.scala:69)
	at scala.meta.internal.metals.MetalsLanguageServer.<init>(MetalsLanguageServer.scala:206)
	at scala.meta.metals.Main$.main(Main.scala:53)
	at scala.meta.metals.Main.main(Main.scala)

2022.03.26 13:04:48 INFO  time: code lens generation in 7.28s
2022.03.26 13:04:48 INFO  time: code lens generation in 1.15s
Mar 26, 2022 1:04:49 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
SEVERE: Internal error: java.util.NoSuchElementException: BatchedFunction
java.util.concurrent.CompletionException: java.util.NoSuchElementException: BatchedFunction
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:292)
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:308)
	at java.util.concurrent.CompletableFuture.uniAccept(CompletableFuture.java:661)
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:646)
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:488)
	at java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:1990)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1(CancelTokens.scala:40)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1$adapted(CancelTokens.scala:38)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.util.NoSuchElementException: BatchedFunction
	at scala.meta.internal.metals.BatchedFunction.<init>(BatchedFunction.scala:69)
	at scala.meta.internal.metals.MetalsLanguageServer.<init>(MetalsLanguageServer.scala:206)
	at scala.meta.metals.Main$.main(Main.scala:53)
	at scala.meta.metals.Main.main(Main.scala)

Mar 26, 2022 1:04:53 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
SEVERE: Internal error: java.util.NoSuchElementException: BatchedFunction
java.util.concurrent.CompletionException: java.util.NoSuchElementException: BatchedFunction
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:292)
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:308)
	at java.util.concurrent.CompletableFuture.uniAccept(CompletableFuture.java:661)
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:646)
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:488)
	at java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:1990)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1(CancelTokens.scala:40)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1$adapted(CancelTokens.scala:38)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.util.NoSuchElementException: BatchedFunction
	at scala.meta.internal.metals.BatchedFunction.<init>(BatchedFunction.scala:69)
	at scala.meta.internal.metals.MetalsLanguageServer.<init>(MetalsLanguageServer.scala:206)
	at scala.meta.metals.Main$.main(Main.scala:53)
	at scala.meta.metals.Main.main(Main.scala)

Mar 26, 2022 1:04:54 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
SEVERE: Internal error: java.util.NoSuchElementException: BatchedFunction
java.util.concurrent.CompletionException: java.util.NoSuchElementException: BatchedFunction
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:292)
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:308)
	at java.util.concurrent.CompletableFuture.uniAccept(CompletableFuture.java:661)
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:646)
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:488)
	at java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:1990)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1(CancelTokens.scala:40)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1$adapted(CancelTokens.scala:38)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.util.NoSuchElementException: BatchedFunction
	at scala.meta.internal.metals.BatchedFunction.<init>(BatchedFunction.scala:69)
	at scala.meta.internal.metals.MetalsLanguageServer.<init>(MetalsLanguageServer.scala:206)
	at scala.meta.metals.Main$.main(Main.scala:53)
	at scala.meta.metals.Main.main(Main.scala)

2022.03.26 13:05:35 ERROR No project associated with Uri(file:/Users/Tarin/Documents/Projects/Programming/spark_testing/project?id=spark_testing-build)
2022.03.26 13:05:35 ERROR No project associated with Uri(file:/Users/Tarin/Documents/Projects/Programming/spark_testing/project?id=spark_testing-build)
2022.03.26 13:05:35 ERROR No project associated with Uri(file:/Users/Tarin/Documents/Projects/Programming/spark_testing/project?id=spark_testing-build)
2022.03.26 13:07:04 WARN  no build target for: /Users/Tarin/Documents/Projects/Programming/flight-data/src/main/scala/FlightData.scala
2022.03.26 13:08:38 ERROR No project associated with Uri(file:/Users/Tarin/Documents/Projects/Programming/spark_testing/project?id=spark_testing-build)
2022.03.26 13:08:38 ERROR No project associated with Uri(file:/Users/Tarin/Documents/Projects/Programming/spark_testing/project?id=spark_testing-build)
2022.03.26 13:08:38 ERROR No project associated with Uri(file:/Users/Tarin/Documents/Projects/Programming/spark_testing/project?id=spark_testing-build)
2022.03.26 13:08:49 ERROR No project associated with Uri(file:/Users/Tarin/Documents/Projects/Programming/spark_testing/project?id=spark_testing-build)
2022.03.26 13:08:49 ERROR No project associated with Uri(file:/Users/Tarin/Documents/Projects/Programming/spark_testing/project?id=spark_testing-build)
2022.03.26 13:08:49 ERROR No project associated with Uri(file:/Users/Tarin/Documents/Projects/Programming/spark_testing/project?id=spark_testing-build)
2022.03.26 13:11:28 WARN  no build target for: /Users/Tarin/Documents/Projects/Programming/flight-data/src/main/scala/FlightData.scala
2022.03.26 13:11:28 WARN  no build target for: /Users/Tarin/Documents/Projects/Programming/flight-data/src/main/scala/FlightData.scala
Mar 26, 2022 1:11:28 PM scala.meta.internal.pc.CompilerAccess handleError
SEVERE: file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:5: error: illegal start of definition val
val spark = SparkSession.builder().appName("").master("local").getOrCreate
^
file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:5: error: illegal start of definition val
val spark = SparkSession.builder().appName("").master("local").getOrCreate
^
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.ScalametaParser.statSeq(ScalametaParser.scala:4156)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$batchSource$1(ScalametaParser.scala:4391)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.batchSource(ScalametaParser.scala:4350)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$source$1(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.source(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.entrypointSource(ScalametaParser.scala:4348)
	at scala.meta.internal.parsers.ScalametaParser.parseSourceImpl(ScalametaParser.scala:132)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$parseSource$1(ScalametaParser.scala:129)
	at scala.meta.internal.parsers.ScalametaParser.parseRuleAfterBOF(ScalametaParser.scala:56)
	at scala.meta.internal.parsers.ScalametaParser.parseRule(ScalametaParser.scala:51)
	at scala.meta.internal.parsers.ScalametaParser.parseSource(ScalametaParser.scala:129)
	at scala.meta.parsers.Parse$.$anonfun$parseSource$1(Parse.scala:29)
	at scala.meta.parsers.Parse$$anon$1.apply(Parse.scala:36)
	at scala.meta.parsers.Api$XtensionParseDialectInput.parse(Api.scala:25)
	at scala.meta.internal.semanticdb.scalac.ParseOps$XtensionCompilationUnitSource.toSource(ParseOps.scala:17)
	at scala.meta.internal.semanticdb.scalac.TextDocumentOps$XtensionCompilationUnitDocument.toTextDocument(TextDocumentOps.scala:201)
	at scala.meta.internal.pc.SemanticdbTextDocumentProvider.textDocument(SemanticdbTextDocumentProvider.scala:54)

Mar 26, 2022 1:11:52 PM scala.meta.internal.pc.CompilerAccess handleError
SEVERE: file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:5: error: illegal start of definition val
val spark = SparkSession.builder().appName("").master("local").getOrCreate
^
file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:5: error: illegal start of definition val
val spark = SparkSession.builder().appName("").master("local").getOrCreate
^
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.ScalametaParser.statSeq(ScalametaParser.scala:4156)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$batchSource$1(ScalametaParser.scala:4391)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.batchSource(ScalametaParser.scala:4350)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$source$1(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.source(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.entrypointSource(ScalametaParser.scala:4348)
	at scala.meta.internal.parsers.ScalametaParser.parseSourceImpl(ScalametaParser.scala:132)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$parseSource$1(ScalametaParser.scala:129)
	at scala.meta.internal.parsers.ScalametaParser.parseRuleAfterBOF(ScalametaParser.scala:56)
	at scala.meta.internal.parsers.ScalametaParser.parseRule(ScalametaParser.scala:51)
	at scala.meta.internal.parsers.ScalametaParser.parseSource(ScalametaParser.scala:129)
	at scala.meta.parsers.Parse$.$anonfun$parseSource$1(Parse.scala:29)
	at scala.meta.parsers.Parse$$anon$1.apply(Parse.scala:36)
	at scala.meta.parsers.Api$XtensionParseDialectInput.parse(Api.scala:25)
	at scala.meta.internal.semanticdb.scalac.ParseOps$XtensionCompilationUnitSource.toSource(ParseOps.scala:17)
	at scala.meta.internal.semanticdb.scalac.TextDocumentOps$XtensionCompilationUnitDocument.toTextDocument(TextDocumentOps.scala:201)
	at scala.meta.internal.pc.SemanticdbTextDocumentProvider.textDocument(SemanticdbTextDocumentProvider.scala:54)

Mar 26, 2022 1:11:53 PM scala.meta.internal.pc.CompilerAccess handleError
SEVERE: file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:5: error: illegal start of definition val
val spark = SparkSession.builder().appName("").master("local").getOrCreate
^
file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:5: error: illegal start of definition val
val spark = SparkSession.builder().appName("").master("local").getOrCreate
^
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.ScalametaParser.statSeq(ScalametaParser.scala:4156)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$batchSource$1(ScalametaParser.scala:4391)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.batchSource(ScalametaParser.scala:4350)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$source$1(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.source(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.entrypointSource(ScalametaParser.scala:4348)
	at scala.meta.internal.parsers.ScalametaParser.parseSourceImpl(ScalametaParser.scala:132)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$parseSource$1(ScalametaParser.scala:129)
	at scala.meta.internal.parsers.ScalametaParser.parseRuleAfterBOF(ScalametaParser.scala:56)
	at scala.meta.internal.parsers.ScalametaParser.parseRule(ScalametaParser.scala:51)
	at scala.meta.internal.parsers.ScalametaParser.parseSource(ScalametaParser.scala:129)
	at scala.meta.parsers.Parse$.$anonfun$parseSource$1(Parse.scala:29)
	at scala.meta.parsers.Parse$$anon$1.apply(Parse.scala:36)
	at scala.meta.parsers.Api$XtensionParseDialectInput.parse(Api.scala:25)
	at scala.meta.internal.semanticdb.scalac.ParseOps$XtensionCompilationUnitSource.toSource(ParseOps.scala:17)
	at scala.meta.internal.semanticdb.scalac.TextDocumentOps$XtensionCompilationUnitDocument.toTextDocument(TextDocumentOps.scala:201)
	at scala.meta.internal.pc.SemanticdbTextDocumentProvider.textDocument(SemanticdbTextDocumentProvider.scala:54)

Mar 26, 2022 1:11:54 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
SEVERE: Internal error: java.lang.reflect.InvocationTargetException
java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
	at org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.lambda$null$0(GenericEndpoint.java:67)
	at org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.request(GenericEndpoint.java:120)
	at org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.handleRequest(RemoteEndpoint.java:261)
	at org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.consume(RemoteEndpoint.java:190)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.handleMessage(StreamMessageProducer.java:194)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.listen(StreamMessageProducer.java:94)
	at org.eclipse.lsp4j.jsonrpc.json.ConcurrentMessageProcessor.run(ConcurrentMessageProcessor.java:113)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.reflect.InvocationTargetException
	at sun.reflect.GeneratedMethodAccessor2.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.lambda$null$0(GenericEndpoint.java:65)
	... 11 more
Caused by: java.lang.IllegalArgumentException: 12 is not a valid line number, allowed [0..11]
	at scala.meta.internal.inputs.InternalInput.lineToOffset(InternalInput.scala:37)
	at scala.meta.internal.inputs.InternalInput.lineToOffset$(InternalInput.scala:32)
	at scala.meta.inputs.Input$VirtualFile.lineToOffset(Input.scala:80)
	at scala.meta.inputs.Position$Range$.lineLength$1(Position.scala:57)
	at scala.meta.inputs.Position$Range$.apply(Position.scala:60)
	at scala.meta.internal.mtags.MtagsEnrichments$XtensionPositionLspInverse.toMeta(MtagsEnrichments.scala:143)
	at scala.meta.internal.parsing.Trees.$anonfun$findLastEnclosingAt$4(Trees.scala:78)
	at scala.Option.flatMap(Option.scala:271)
	at scala.meta.internal.parsing.Trees.findLastEnclosingAt(Trees.scala:77)
	at scala.meta.internal.metals.codeactions.PatternMatchRefactor.contribute(PatternMatchRefactor.scala:96)
	at scala.meta.internal.metals.CodeActionProvider$$anonfun$1.applyOrElse(CodeActionProvider.scala:64)
	at scala.meta.internal.metals.CodeActionProvider$$anonfun$1.applyOrElse(CodeActionProvider.scala:63)
	at scala.collection.immutable.List.collect(List.scala:323)
	at scala.meta.internal.metals.CodeActionProvider.codeActions(CodeActionProvider.scala:63)
	at scala.meta.internal.metals.MetalsLanguageServer.$anonfun$codeAction$1(MetalsLanguageServer.scala:1615)
	at scala.meta.internal.metals.CancelTokens$.future(CancelTokens.scala:38)
	at scala.meta.internal.metals.MetalsLanguageServer.codeAction(MetalsLanguageServer.scala:1614)
	... 15 more

2022.03.26 13:11:54 WARN  no build target for: /Users/Tarin/Documents/Projects/Programming/flight-data/src/main/scala/FlightData.scala
Mar 26, 2022 1:11:54 PM scala.meta.internal.pc.CompilerAccess handleError
SEVERE: file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:5: error: illegal start of definition val
val spark = SparkSession.builder().appName("testing").master("local").getOrCreate
^
file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:5: error: illegal start of definition val
val spark = SparkSession.builder().appName("testing").master("local").getOrCreate
^
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.ScalametaParser.statSeq(ScalametaParser.scala:4156)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$batchSource$1(ScalametaParser.scala:4391)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.batchSource(ScalametaParser.scala:4350)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$source$1(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.source(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.entrypointSource(ScalametaParser.scala:4348)
	at scala.meta.internal.parsers.ScalametaParser.parseSourceImpl(ScalametaParser.scala:132)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$parseSource$1(ScalametaParser.scala:129)
	at scala.meta.internal.parsers.ScalametaParser.parseRuleAfterBOF(ScalametaParser.scala:56)
	at scala.meta.internal.parsers.ScalametaParser.parseRule(ScalametaParser.scala:51)
	at scala.meta.internal.parsers.ScalametaParser.parseSource(ScalametaParser.scala:129)
	at scala.meta.parsers.Parse$.$anonfun$parseSource$1(Parse.scala:29)
	at scala.meta.parsers.Parse$$anon$1.apply(Parse.scala:36)
	at scala.meta.parsers.Api$XtensionParseDialectInput.parse(Api.scala:25)
	at scala.meta.internal.semanticdb.scalac.ParseOps$XtensionCompilationUnitSource.toSource(ParseOps.scala:17)
	at scala.meta.internal.semanticdb.scalac.TextDocumentOps$XtensionCompilationUnitDocument.toTextDocument(TextDocumentOps.scala:201)
	at scala.meta.internal.pc.SemanticdbTextDocumentProvider.textDocument(SemanticdbTextDocumentProvider.scala:54)

Mar 26, 2022 1:12:29 PM scala.meta.internal.pc.CompilerAccess handleError
SEVERE: file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:5: error: illegal start of definition val
val spark = SparkSession.builder().appName("testing").master("local").getOrCreate
^
file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:5: error: illegal start of definition val
val spark = SparkSession.builder().appName("testing").master("local").getOrCreate
^
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.ScalametaParser.statSeq(ScalametaParser.scala:4156)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$batchSource$1(ScalametaParser.scala:4391)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.batchSource(ScalametaParser.scala:4350)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$source$1(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.source(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.entrypointSource(ScalametaParser.scala:4348)
	at scala.meta.internal.parsers.ScalametaParser.parseSourceImpl(ScalametaParser.scala:132)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$parseSource$1(ScalametaParser.scala:129)
	at scala.meta.internal.parsers.ScalametaParser.parseRuleAfterBOF(ScalametaParser.scala:56)
	at scala.meta.internal.parsers.ScalametaParser.parseRule(ScalametaParser.scala:51)
	at scala.meta.internal.parsers.ScalametaParser.parseSource(ScalametaParser.scala:129)
	at scala.meta.parsers.Parse$.$anonfun$parseSource$1(Parse.scala:29)
	at scala.meta.parsers.Parse$$anon$1.apply(Parse.scala:36)
	at scala.meta.parsers.Api$XtensionParseDialectInput.parse(Api.scala:25)
	at scala.meta.internal.semanticdb.scalac.ParseOps$XtensionCompilationUnitSource.toSource(ParseOps.scala:17)
	at scala.meta.internal.semanticdb.scalac.TextDocumentOps$XtensionCompilationUnitDocument.toTextDocument(TextDocumentOps.scala:201)
	at scala.meta.internal.pc.SemanticdbTextDocumentProvider.textDocument(SemanticdbTextDocumentProvider.scala:54)

Mar 26, 2022 1:12:29 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
SEVERE: Internal error: java.lang.reflect.InvocationTargetException
java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
	at org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.lambda$null$0(GenericEndpoint.java:67)
	at org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.request(GenericEndpoint.java:120)
	at org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.handleRequest(RemoteEndpoint.java:261)
	at org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.consume(RemoteEndpoint.java:190)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.handleMessage(StreamMessageProducer.java:194)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.listen(StreamMessageProducer.java:94)
	at org.eclipse.lsp4j.jsonrpc.json.ConcurrentMessageProcessor.run(ConcurrentMessageProcessor.java:113)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.reflect.InvocationTargetException
	at sun.reflect.GeneratedMethodAccessor2.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.lambda$null$0(GenericEndpoint.java:65)
	... 11 more
Caused by: java.lang.IllegalArgumentException: 12 is not a valid line number, allowed [0..11]
	at scala.meta.internal.inputs.InternalInput.lineToOffset(InternalInput.scala:37)
	at scala.meta.internal.inputs.InternalInput.lineToOffset$(InternalInput.scala:32)
	at scala.meta.inputs.Input$VirtualFile.lineToOffset(Input.scala:80)
	at scala.meta.inputs.Position$Range$.lineLength$1(Position.scala:57)
	at scala.meta.inputs.Position$Range$.apply(Position.scala:60)
	at scala.meta.internal.mtags.MtagsEnrichments$XtensionPositionLspInverse.toMeta(MtagsEnrichments.scala:143)
	at scala.meta.internal.parsing.Trees.$anonfun$findLastEnclosingAt$4(Trees.scala:78)
	at scala.Option.flatMap(Option.scala:271)
	at scala.meta.internal.parsing.Trees.findLastEnclosingAt(Trees.scala:77)
	at scala.meta.internal.metals.codeactions.PatternMatchRefactor.contribute(PatternMatchRefactor.scala:96)
	at scala.meta.internal.metals.CodeActionProvider$$anonfun$1.applyOrElse(CodeActionProvider.scala:64)
	at scala.meta.internal.metals.CodeActionProvider$$anonfun$1.applyOrElse(CodeActionProvider.scala:63)
	at scala.collection.immutable.List.collect(List.scala:323)
	at scala.meta.internal.metals.CodeActionProvider.codeActions(CodeActionProvider.scala:63)
	at scala.meta.internal.metals.MetalsLanguageServer.$anonfun$codeAction$1(MetalsLanguageServer.scala:1615)
	at scala.meta.internal.metals.CancelTokens$.future(CancelTokens.scala:38)
	at scala.meta.internal.metals.MetalsLanguageServer.codeAction(MetalsLanguageServer.scala:1614)
	... 15 more

Mar 26, 2022 1:12:30 PM scala.meta.internal.pc.CompilerAccess handleError
SEVERE: file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:5: error: illegal start of definition val
val spark = SparkSession.builder().appName("testing").master("local").getOrCreate
^
file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:5: error: illegal start of definition val
val spark = SparkSession.builder().appName("testing").master("local").getOrCreate
^
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.ScalametaParser.statSeq(ScalametaParser.scala:4156)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$batchSource$1(ScalametaParser.scala:4391)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.batchSource(ScalametaParser.scala:4350)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$source$1(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.source(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.entrypointSource(ScalametaParser.scala:4348)
	at scala.meta.internal.parsers.ScalametaParser.parseSourceImpl(ScalametaParser.scala:132)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$parseSource$1(ScalametaParser.scala:129)
	at scala.meta.internal.parsers.ScalametaParser.parseRuleAfterBOF(ScalametaParser.scala:56)
	at scala.meta.internal.parsers.ScalametaParser.parseRule(ScalametaParser.scala:51)
	at scala.meta.internal.parsers.ScalametaParser.parseSource(ScalametaParser.scala:129)
	at scala.meta.parsers.Parse$.$anonfun$parseSource$1(Parse.scala:29)
	at scala.meta.parsers.Parse$$anon$1.apply(Parse.scala:36)
	at scala.meta.parsers.Api$XtensionParseDialectInput.parse(Api.scala:25)
	at scala.meta.internal.semanticdb.scalac.ParseOps$XtensionCompilationUnitSource.toSource(ParseOps.scala:17)
	at scala.meta.internal.semanticdb.scalac.TextDocumentOps$XtensionCompilationUnitDocument.toTextDocument(TextDocumentOps.scala:201)
	at scala.meta.internal.pc.SemanticdbTextDocumentProvider.textDocument(SemanticdbTextDocumentProvider.scala:54)

Mar 26, 2022 1:12:30 PM scala.meta.internal.pc.CompilerAccess handleError
SEVERE: file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:5: error: illegal start of definition val
val spark = SparkSession.builder().appName("testing").master("local").getOrCreate
^
file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:5: error: illegal start of definition val
val spark = SparkSession.builder().appName("testing").master("local").getOrCreate
^
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.ScalametaParser.statSeq(ScalametaParser.scala:4156)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$batchSource$1(ScalametaParser.scala:4391)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.batchSource(ScalametaParser.scala:4350)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$source$1(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.source(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.entrypointSource(ScalametaParser.scala:4348)
	at scala.meta.internal.parsers.ScalametaParser.parseSourceImpl(ScalametaParser.scala:132)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$parseSource$1(ScalametaParser.scala:129)
	at scala.meta.internal.parsers.ScalametaParser.parseRuleAfterBOF(ScalametaParser.scala:56)
	at scala.meta.internal.parsers.ScalametaParser.parseRule(ScalametaParser.scala:51)
	at scala.meta.internal.parsers.ScalametaParser.parseSource(ScalametaParser.scala:129)
	at scala.meta.parsers.Parse$.$anonfun$parseSource$1(Parse.scala:29)
	at scala.meta.parsers.Parse$$anon$1.apply(Parse.scala:36)
	at scala.meta.parsers.Api$XtensionParseDialectInput.parse(Api.scala:25)
	at scala.meta.internal.semanticdb.scalac.ParseOps$XtensionCompilationUnitSource.toSource(ParseOps.scala:17)
	at scala.meta.internal.semanticdb.scalac.TextDocumentOps$XtensionCompilationUnitDocument.toTextDocument(TextDocumentOps.scala:201)
	at scala.meta.internal.pc.SemanticdbTextDocumentProvider.textDocument(SemanticdbTextDocumentProvider.scala:54)

Mar 26, 2022 1:12:30 PM scala.meta.internal.pc.CompilerAccess handleError
SEVERE: file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:5: error: illegal start of definition val
val spark = SparkSession.builder().appName("testing").master("local").getOrCreate
^
file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:5: error: illegal start of definition val
val spark = SparkSession.builder().appName("testing").master("local").getOrCreate
^
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.ScalametaParser.statSeq(ScalametaParser.scala:4156)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$batchSource$1(ScalametaParser.scala:4391)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.batchSource(ScalametaParser.scala:4350)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$source$1(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.source(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.entrypointSource(ScalametaParser.scala:4348)
	at scala.meta.internal.parsers.ScalametaParser.parseSourceImpl(ScalametaParser.scala:132)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$parseSource$1(ScalametaParser.scala:129)
	at scala.meta.internal.parsers.ScalametaParser.parseRuleAfterBOF(ScalametaParser.scala:56)
	at scala.meta.internal.parsers.ScalametaParser.parseRule(ScalametaParser.scala:51)
	at scala.meta.internal.parsers.ScalametaParser.parseSource(ScalametaParser.scala:129)
	at scala.meta.parsers.Parse$.$anonfun$parseSource$1(Parse.scala:29)
	at scala.meta.parsers.Parse$$anon$1.apply(Parse.scala:36)
	at scala.meta.parsers.Api$XtensionParseDialectInput.parse(Api.scala:25)
	at scala.meta.internal.semanticdb.scalac.ParseOps$XtensionCompilationUnitSource.toSource(ParseOps.scala:17)
	at scala.meta.internal.semanticdb.scalac.TextDocumentOps$XtensionCompilationUnitDocument.toTextDocument(TextDocumentOps.scala:201)
	at scala.meta.internal.pc.SemanticdbTextDocumentProvider.textDocument(SemanticdbTextDocumentProvider.scala:54)

Mar 26, 2022 1:12:32 PM scala.meta.internal.pc.CompilerAccess handleError
SEVERE: file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:5: error: illegal start of definition val
val spark = SparkSession.builder().appName("testing").master("local").getOrCreate
^
file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:5: error: illegal start of definition val
val spark = SparkSession.builder().appName("testing").master("local").getOrCreate
^
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.ScalametaParser.statSeq(ScalametaParser.scala:4156)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$batchSource$1(ScalametaParser.scala:4391)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.batchSource(ScalametaParser.scala:4350)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$source$1(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.source(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.entrypointSource(ScalametaParser.scala:4348)
	at scala.meta.internal.parsers.ScalametaParser.parseSourceImpl(ScalametaParser.scala:132)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$parseSource$1(ScalametaParser.scala:129)
	at scala.meta.internal.parsers.ScalametaParser.parseRuleAfterBOF(ScalametaParser.scala:56)
	at scala.meta.internal.parsers.ScalametaParser.parseRule(ScalametaParser.scala:51)
	at scala.meta.internal.parsers.ScalametaParser.parseSource(ScalametaParser.scala:129)
	at scala.meta.parsers.Parse$.$anonfun$parseSource$1(Parse.scala:29)
	at scala.meta.parsers.Parse$$anon$1.apply(Parse.scala:36)
	at scala.meta.parsers.Api$XtensionParseDialectInput.parse(Api.scala:25)
	at scala.meta.internal.semanticdb.scalac.ParseOps$XtensionCompilationUnitSource.toSource(ParseOps.scala:17)
	at scala.meta.internal.semanticdb.scalac.TextDocumentOps$XtensionCompilationUnitDocument.toTextDocument(TextDocumentOps.scala:201)
	at scala.meta.internal.pc.SemanticdbTextDocumentProvider.textDocument(SemanticdbTextDocumentProvider.scala:54)

Mar 26, 2022 1:13:00 PM scala.meta.internal.pc.CompilerAccess handleError
SEVERE: file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:5: error: illegal start of definition val
val spark = SparkSession.builder().appName("testing").master("local").getOrCreate
^
file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:5: error: illegal start of definition val
val spark = SparkSession.builder().appName("testing").master("local").getOrCreate
^
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.ScalametaParser.statSeq(ScalametaParser.scala:4156)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$batchSource$1(ScalametaParser.scala:4391)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.batchSource(ScalametaParser.scala:4350)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$source$1(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.source(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.entrypointSource(ScalametaParser.scala:4348)
	at scala.meta.internal.parsers.ScalametaParser.parseSourceImpl(ScalametaParser.scala:132)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$parseSource$1(ScalametaParser.scala:129)
	at scala.meta.internal.parsers.ScalametaParser.parseRuleAfterBOF(ScalametaParser.scala:56)
	at scala.meta.internal.parsers.ScalametaParser.parseRule(ScalametaParser.scala:51)
	at scala.meta.internal.parsers.ScalametaParser.parseSource(ScalametaParser.scala:129)
	at scala.meta.parsers.Parse$.$anonfun$parseSource$1(Parse.scala:29)
	at scala.meta.parsers.Parse$$anon$1.apply(Parse.scala:36)
	at scala.meta.parsers.Api$XtensionParseDialectInput.parse(Api.scala:25)
	at scala.meta.internal.semanticdb.scalac.ParseOps$XtensionCompilationUnitSource.toSource(ParseOps.scala:17)
	at scala.meta.internal.semanticdb.scalac.TextDocumentOps$XtensionCompilationUnitDocument.toTextDocument(TextDocumentOps.scala:201)
	at scala.meta.internal.pc.SemanticdbTextDocumentProvider.textDocument(SemanticdbTextDocumentProvider.scala:54)

Mar 26, 2022 1:13:02 PM scala.meta.internal.pc.CompilerAccess handleError
SEVERE: file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:5: error: illegal start of definition val
val spark = SparkSession.builder().appName("testing").master("local").getOrCreate
^
file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:5: error: illegal start of definition val
val spark = SparkSession.builder().appName("testing").master("local").getOrCreate
^
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.ScalametaParser.statSeq(ScalametaParser.scala:4156)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$batchSource$1(ScalametaParser.scala:4391)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.batchSource(ScalametaParser.scala:4350)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$source$1(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.source(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.entrypointSource(ScalametaParser.scala:4348)
	at scala.meta.internal.parsers.ScalametaParser.parseSourceImpl(ScalametaParser.scala:132)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$parseSource$1(ScalametaParser.scala:129)
	at scala.meta.internal.parsers.ScalametaParser.parseRuleAfterBOF(ScalametaParser.scala:56)
	at scala.meta.internal.parsers.ScalametaParser.parseRule(ScalametaParser.scala:51)
	at scala.meta.internal.parsers.ScalametaParser.parseSource(ScalametaParser.scala:129)
	at scala.meta.parsers.Parse$.$anonfun$parseSource$1(Parse.scala:29)
	at scala.meta.parsers.Parse$$anon$1.apply(Parse.scala:36)
	at scala.meta.parsers.Api$XtensionParseDialectInput.parse(Api.scala:25)
	at scala.meta.internal.semanticdb.scalac.ParseOps$XtensionCompilationUnitSource.toSource(ParseOps.scala:17)
	at scala.meta.internal.semanticdb.scalac.TextDocumentOps$XtensionCompilationUnitDocument.toTextDocument(TextDocumentOps.scala:201)
	at scala.meta.internal.pc.SemanticdbTextDocumentProvider.textDocument(SemanticdbTextDocumentProvider.scala:54)

Mar 26, 2022 1:13:02 PM scala.meta.internal.pc.CompilerAccess handleError
SEVERE: file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:5: error: illegal start of definition val
val spark = SparkSession.builder().appName("testing").master("local").getOrCreate
^
file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:5: error: illegal start of definition val
val spark = SparkSession.builder().appName("testing").master("local").getOrCreate
^
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.ScalametaParser.statSeq(ScalametaParser.scala:4156)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$batchSource$1(ScalametaParser.scala:4391)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.batchSource(ScalametaParser.scala:4350)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$source$1(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.source(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.entrypointSource(ScalametaParser.scala:4348)
	at scala.meta.internal.parsers.ScalametaParser.parseSourceImpl(ScalametaParser.scala:132)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$parseSource$1(ScalametaParser.scala:129)
	at scala.meta.internal.parsers.ScalametaParser.parseRuleAfterBOF(ScalametaParser.scala:56)
	at scala.meta.internal.parsers.ScalametaParser.parseRule(ScalametaParser.scala:51)
	at scala.meta.internal.parsers.ScalametaParser.parseSource(ScalametaParser.scala:129)
	at scala.meta.parsers.Parse$.$anonfun$parseSource$1(Parse.scala:29)
	at scala.meta.parsers.Parse$$anon$1.apply(Parse.scala:36)
	at scala.meta.parsers.Api$XtensionParseDialectInput.parse(Api.scala:25)
	at scala.meta.internal.semanticdb.scalac.ParseOps$XtensionCompilationUnitSource.toSource(ParseOps.scala:17)
	at scala.meta.internal.semanticdb.scalac.TextDocumentOps$XtensionCompilationUnitDocument.toTextDocument(TextDocumentOps.scala:201)
	at scala.meta.internal.pc.SemanticdbTextDocumentProvider.textDocument(SemanticdbTextDocumentProvider.scala:54)

Mar 26, 2022 1:13:03 PM scala.meta.internal.pc.CompilerAccess handleError
SEVERE: file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:5: error: illegal start of definition val
val spark = SparkSession.builder().appName("testing").master("local").getOrCreate
^
file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:5: error: illegal start of definition val
val spark = SparkSession.builder().appName("testing").master("local").getOrCreate
^
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.ScalametaParser.statSeq(ScalametaParser.scala:4156)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$batchSource$1(ScalametaParser.scala:4391)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.batchSource(ScalametaParser.scala:4350)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$source$1(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.source(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.entrypointSource(ScalametaParser.scala:4348)
	at scala.meta.internal.parsers.ScalametaParser.parseSourceImpl(ScalametaParser.scala:132)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$parseSource$1(ScalametaParser.scala:129)
	at scala.meta.internal.parsers.ScalametaParser.parseRuleAfterBOF(ScalametaParser.scala:56)
	at scala.meta.internal.parsers.ScalametaParser.parseRule(ScalametaParser.scala:51)
	at scala.meta.internal.parsers.ScalametaParser.parseSource(ScalametaParser.scala:129)
	at scala.meta.parsers.Parse$.$anonfun$parseSource$1(Parse.scala:29)
	at scala.meta.parsers.Parse$$anon$1.apply(Parse.scala:36)
	at scala.meta.parsers.Api$XtensionParseDialectInput.parse(Api.scala:25)
	at scala.meta.internal.semanticdb.scalac.ParseOps$XtensionCompilationUnitSource.toSource(ParseOps.scala:17)
	at scala.meta.internal.semanticdb.scalac.TextDocumentOps$XtensionCompilationUnitDocument.toTextDocument(TextDocumentOps.scala:201)
	at scala.meta.internal.pc.SemanticdbTextDocumentProvider.textDocument(SemanticdbTextDocumentProvider.scala:54)

Mar 26, 2022 1:13:06 PM scala.meta.internal.pc.CompilerAccess handleError
SEVERE: file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:5: error: illegal start of definition val
val spark = SparkSession.builder().appName("testing").master("local").getOrCreate
^
file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:5: error: illegal start of definition val
val spark = SparkSession.builder().appName("testing").master("local").getOrCreate
^
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.ScalametaParser.statSeq(ScalametaParser.scala:4156)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$batchSource$1(ScalametaParser.scala:4391)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.batchSource(ScalametaParser.scala:4350)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$source$1(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.source(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.entrypointSource(ScalametaParser.scala:4348)
	at scala.meta.internal.parsers.ScalametaParser.parseSourceImpl(ScalametaParser.scala:132)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$parseSource$1(ScalametaParser.scala:129)
	at scala.meta.internal.parsers.ScalametaParser.parseRuleAfterBOF(ScalametaParser.scala:56)
	at scala.meta.internal.parsers.ScalametaParser.parseRule(ScalametaParser.scala:51)
	at scala.meta.internal.parsers.ScalametaParser.parseSource(ScalametaParser.scala:129)
	at scala.meta.parsers.Parse$.$anonfun$parseSource$1(Parse.scala:29)
	at scala.meta.parsers.Parse$$anon$1.apply(Parse.scala:36)
	at scala.meta.parsers.Api$XtensionParseDialectInput.parse(Api.scala:25)
	at scala.meta.internal.semanticdb.scalac.ParseOps$XtensionCompilationUnitSource.toSource(ParseOps.scala:17)
	at scala.meta.internal.semanticdb.scalac.TextDocumentOps$XtensionCompilationUnitDocument.toTextDocument(TextDocumentOps.scala:201)
	at scala.meta.internal.pc.SemanticdbTextDocumentProvider.textDocument(SemanticdbTextDocumentProvider.scala:54)

Mar 26, 2022 1:13:08 PM scala.meta.internal.pc.CompilerAccess handleError
SEVERE: file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:5: error: illegal start of definition val
val spark = SparkSession.builder().appName("testing").master("local").getOrCreate
^
file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:5: error: illegal start of definition val
val spark = SparkSession.builder().appName("testing").master("local").getOrCreate
^
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.ScalametaParser.statSeq(ScalametaParser.scala:4156)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$batchSource$1(ScalametaParser.scala:4391)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.batchSource(ScalametaParser.scala:4350)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$source$1(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.source(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.entrypointSource(ScalametaParser.scala:4348)
	at scala.meta.internal.parsers.ScalametaParser.parseSourceImpl(ScalametaParser.scala:132)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$parseSource$1(ScalametaParser.scala:129)
	at scala.meta.internal.parsers.ScalametaParser.parseRuleAfterBOF(ScalametaParser.scala:56)
	at scala.meta.internal.parsers.ScalametaParser.parseRule(ScalametaParser.scala:51)
	at scala.meta.internal.parsers.ScalametaParser.parseSource(ScalametaParser.scala:129)
	at scala.meta.parsers.Parse$.$anonfun$parseSource$1(Parse.scala:29)
	at scala.meta.parsers.Parse$$anon$1.apply(Parse.scala:36)
	at scala.meta.parsers.Api$XtensionParseDialectInput.parse(Api.scala:25)
	at scala.meta.internal.semanticdb.scalac.ParseOps$XtensionCompilationUnitSource.toSource(ParseOps.scala:17)
	at scala.meta.internal.semanticdb.scalac.TextDocumentOps$XtensionCompilationUnitDocument.toTextDocument(TextDocumentOps.scala:201)
	at scala.meta.internal.pc.SemanticdbTextDocumentProvider.textDocument(SemanticdbTextDocumentProvider.scala:54)

Mar 26, 2022 1:13:09 PM scala.meta.internal.pc.CompilerAccess handleError
SEVERE: file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:5: error: illegal start of definition val
val spark = SparkSession.builder().appName("testing").master("local").getOrCreate
^
file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:5: error: illegal start of definition val
val spark = SparkSession.builder().appName("testing").master("local").getOrCreate
^
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.ScalametaParser.statSeq(ScalametaParser.scala:4156)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$batchSource$1(ScalametaParser.scala:4391)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.batchSource(ScalametaParser.scala:4350)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$source$1(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.source(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.entrypointSource(ScalametaParser.scala:4348)
	at scala.meta.internal.parsers.ScalametaParser.parseSourceImpl(ScalametaParser.scala:132)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$parseSource$1(ScalametaParser.scala:129)
	at scala.meta.internal.parsers.ScalametaParser.parseRuleAfterBOF(ScalametaParser.scala:56)
	at scala.meta.internal.parsers.ScalametaParser.parseRule(ScalametaParser.scala:51)
	at scala.meta.internal.parsers.ScalametaParser.parseSource(ScalametaParser.scala:129)
	at scala.meta.parsers.Parse$.$anonfun$parseSource$1(Parse.scala:29)
	at scala.meta.parsers.Parse$$anon$1.apply(Parse.scala:36)
	at scala.meta.parsers.Api$XtensionParseDialectInput.parse(Api.scala:25)
	at scala.meta.internal.semanticdb.scalac.ParseOps$XtensionCompilationUnitSource.toSource(ParseOps.scala:17)
	at scala.meta.internal.semanticdb.scalac.TextDocumentOps$XtensionCompilationUnitDocument.toTextDocument(TextDocumentOps.scala:201)
	at scala.meta.internal.pc.SemanticdbTextDocumentProvider.textDocument(SemanticdbTextDocumentProvider.scala:54)

Mar 26, 2022 1:13:10 PM scala.meta.internal.pc.CompilerAccess handleError
SEVERE: file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:5: error: illegal start of definition val
val spark = SparkSession.builder().appName("testing").master("local").getOrCreate
^
file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:5: error: illegal start of definition val
val spark = SparkSession.builder().appName("testing").master("local").getOrCreate
^
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.ScalametaParser.statSeq(ScalametaParser.scala:4156)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$batchSource$1(ScalametaParser.scala:4391)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.batchSource(ScalametaParser.scala:4350)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$source$1(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.source(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.entrypointSource(ScalametaParser.scala:4348)
	at scala.meta.internal.parsers.ScalametaParser.parseSourceImpl(ScalametaParser.scala:132)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$parseSource$1(ScalametaParser.scala:129)
	at scala.meta.internal.parsers.ScalametaParser.parseRuleAfterBOF(ScalametaParser.scala:56)
	at scala.meta.internal.parsers.ScalametaParser.parseRule(ScalametaParser.scala:51)
	at scala.meta.internal.parsers.ScalametaParser.parseSource(ScalametaParser.scala:129)
	at scala.meta.parsers.Parse$.$anonfun$parseSource$1(Parse.scala:29)
	at scala.meta.parsers.Parse$$anon$1.apply(Parse.scala:36)
	at scala.meta.parsers.Api$XtensionParseDialectInput.parse(Api.scala:25)
	at scala.meta.internal.semanticdb.scalac.ParseOps$XtensionCompilationUnitSource.toSource(ParseOps.scala:17)
	at scala.meta.internal.semanticdb.scalac.TextDocumentOps$XtensionCompilationUnitDocument.toTextDocument(TextDocumentOps.scala:201)
	at scala.meta.internal.pc.SemanticdbTextDocumentProvider.textDocument(SemanticdbTextDocumentProvider.scala:54)

Mar 26, 2022 1:13:10 PM scala.meta.internal.pc.CompilerAccess handleError
SEVERE: file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:5: error: illegal start of definition val
val spark = SparkSession.builder().appName("testing").master("local").getOrCreate
^
file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:5: error: illegal start of definition val
val spark = SparkSession.builder().appName("testing").master("local").getOrCreate
^
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.ScalametaParser.statSeq(ScalametaParser.scala:4156)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$batchSource$1(ScalametaParser.scala:4391)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.batchSource(ScalametaParser.scala:4350)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$source$1(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.source(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.entrypointSource(ScalametaParser.scala:4348)
	at scala.meta.internal.parsers.ScalametaParser.parseSourceImpl(ScalametaParser.scala:132)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$parseSource$1(ScalametaParser.scala:129)
	at scala.meta.internal.parsers.ScalametaParser.parseRuleAfterBOF(ScalametaParser.scala:56)
	at scala.meta.internal.parsers.ScalametaParser.parseRule(ScalametaParser.scala:51)
	at scala.meta.internal.parsers.ScalametaParser.parseSource(ScalametaParser.scala:129)
	at scala.meta.parsers.Parse$.$anonfun$parseSource$1(Parse.scala:29)
	at scala.meta.parsers.Parse$$anon$1.apply(Parse.scala:36)
	at scala.meta.parsers.Api$XtensionParseDialectInput.parse(Api.scala:25)
	at scala.meta.internal.semanticdb.scalac.ParseOps$XtensionCompilationUnitSource.toSource(ParseOps.scala:17)
	at scala.meta.internal.semanticdb.scalac.TextDocumentOps$XtensionCompilationUnitDocument.toTextDocument(TextDocumentOps.scala:201)
	at scala.meta.internal.pc.SemanticdbTextDocumentProvider.textDocument(SemanticdbTextDocumentProvider.scala:54)

Mar 26, 2022 1:13:10 PM scala.meta.internal.pc.CompilerAccess handleError
SEVERE: file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:5: error: illegal start of definition val
val spark = SparkSession.builder().appName("testing").master("local").getOrCreate
^
file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:5: error: illegal start of definition val
val spark = SparkSession.builder().appName("testing").master("local").getOrCreate
^
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.ScalametaParser.statSeq(ScalametaParser.scala:4156)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$batchSource$1(ScalametaParser.scala:4391)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.batchSource(ScalametaParser.scala:4350)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$source$1(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.source(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.entrypointSource(ScalametaParser.scala:4348)
	at scala.meta.internal.parsers.ScalametaParser.parseSourceImpl(ScalametaParser.scala:132)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$parseSource$1(ScalametaParser.scala:129)
	at scala.meta.internal.parsers.ScalametaParser.parseRuleAfterBOF(ScalametaParser.scala:56)
	at scala.meta.internal.parsers.ScalametaParser.parseRule(ScalametaParser.scala:51)
	at scala.meta.internal.parsers.ScalametaParser.parseSource(ScalametaParser.scala:129)
	at scala.meta.parsers.Parse$.$anonfun$parseSource$1(Parse.scala:29)
	at scala.meta.parsers.Parse$$anon$1.apply(Parse.scala:36)
	at scala.meta.parsers.Api$XtensionParseDialectInput.parse(Api.scala:25)
	at scala.meta.internal.semanticdb.scalac.ParseOps$XtensionCompilationUnitSource.toSource(ParseOps.scala:17)
	at scala.meta.internal.semanticdb.scalac.TextDocumentOps$XtensionCompilationUnitDocument.toTextDocument(TextDocumentOps.scala:201)
	at scala.meta.internal.pc.SemanticdbTextDocumentProvider.textDocument(SemanticdbTextDocumentProvider.scala:54)

Mar 26, 2022 1:13:11 PM scala.meta.internal.pc.CompilerAccess handleError
SEVERE: file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:5: error: illegal start of definition val
val spark = SparkSession.builder().appName("testing").master("local").getOrCreate
^
file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:5: error: illegal start of definition val
val spark = SparkSession.builder().appName("testing").master("local").getOrCreate
^
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.ScalametaParser.statSeq(ScalametaParser.scala:4156)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$batchSource$1(ScalametaParser.scala:4391)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.batchSource(ScalametaParser.scala:4350)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$source$1(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.source(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.entrypointSource(ScalametaParser.scala:4348)
	at scala.meta.internal.parsers.ScalametaParser.parseSourceImpl(ScalametaParser.scala:132)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$parseSource$1(ScalametaParser.scala:129)
	at scala.meta.internal.parsers.ScalametaParser.parseRuleAfterBOF(ScalametaParser.scala:56)
	at scala.meta.internal.parsers.ScalametaParser.parseRule(ScalametaParser.scala:51)
	at scala.meta.internal.parsers.ScalametaParser.parseSource(ScalametaParser.scala:129)
	at scala.meta.parsers.Parse$.$anonfun$parseSource$1(Parse.scala:29)
	at scala.meta.parsers.Parse$$anon$1.apply(Parse.scala:36)
	at scala.meta.parsers.Api$XtensionParseDialectInput.parse(Api.scala:25)
	at scala.meta.internal.semanticdb.scalac.ParseOps$XtensionCompilationUnitSource.toSource(ParseOps.scala:17)
	at scala.meta.internal.semanticdb.scalac.TextDocumentOps$XtensionCompilationUnitDocument.toTextDocument(TextDocumentOps.scala:201)
	at scala.meta.internal.pc.SemanticdbTextDocumentProvider.textDocument(SemanticdbTextDocumentProvider.scala:54)

2022.03.26 13:13:11 WARN  no build target for: /Users/Tarin/Documents/Projects/Programming/flight-data/src/main/scala/FlightData.scala
2022.03.26 13:20:28 WARN  no build target for: /Users/Tarin/Documents/Projects/Programming/flight-data/src/main/scala/FlightData.scala
Mar 26, 2022 1:20:55 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
WARNING: Unmatched cancel notification for request id 201
Mar 26, 2022 1:20:55 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
WARNING: Unmatched cancel notification for request id 202
Mar 26, 2022 1:21:10 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
WARNING: Unmatched cancel notification for request id 241
2022.03.26 13:21:14 WARN  no build target for: /Users/Tarin/Documents/Projects/Programming/flight-data/src/main/scala/FlightData.scala
2022.03.26 13:23:08 WARN  no build target for: /Users/Tarin/Documents/Projects/Programming/flight-data/src/main/scala/FlightData.scala
2022.03.26 13:25:02 WARN  no build target for: /Users/Tarin/Documents/Projects/Programming/flight-data/src/main/scala/FlightData.scala
2022.03.26 13:25:50 WARN  no build target for: /Users/Tarin/Documents/Projects/Programming/flight-data/src/main/scala/FlightData.scala
Mar 26, 2022 1:25:50 PM scala.meta.internal.pc.CompilerAccess handleError
SEVERE: file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:2: error: ; expected but double constant found
import org.apache.spark3.2.1.SparkContext
                        ^
file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:2: error: ; expected but double constant found
import org.apache.spark3.2.1.SparkContext
                        ^
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.ScalametaParser.syntaxErrorExpected(ScalametaParser.scala:334)
	at scala.meta.internal.parsers.ScalametaParser.accept(ScalametaParser.scala:340)
	at scala.meta.internal.parsers.ScalametaParser.acceptStatSep(ScalametaParser.scala:353)
	at scala.meta.internal.parsers.ScalametaParser.acceptStatSepOpt(ScalametaParser.scala:357)
	at scala.meta.internal.parsers.ScalametaParser.statSeq(ScalametaParser.scala:4157)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$batchSource$1(ScalametaParser.scala:4391)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.batchSource(ScalametaParser.scala:4350)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$source$1(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.source(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.entrypointSource(ScalametaParser.scala:4348)
	at scala.meta.internal.parsers.ScalametaParser.parseSourceImpl(ScalametaParser.scala:132)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$parseSource$1(ScalametaParser.scala:129)
	at scala.meta.internal.parsers.ScalametaParser.parseRuleAfterBOF(ScalametaParser.scala:56)
	at scala.meta.internal.parsers.ScalametaParser.parseRule(ScalametaParser.scala:51)
	at scala.meta.internal.parsers.ScalametaParser.parseSource(ScalametaParser.scala:129)
	at scala.meta.parsers.Parse$.$anonfun$parseSource$1(Parse.scala:29)
	at scala.meta.parsers.Parse$$anon$1.apply(Parse.scala:36)
	at scala.meta.parsers.Api$XtensionParseDialectInput.parse(Api.scala:25)
	at scala.meta.internal.semanticdb.scalac.ParseOps$XtensionCompilationUnitSource.toSource(ParseOps.scala:17)
	at scala.meta.internal.semanticdb.scalac.TextDocumentOps$XtensionCompilationUnitDocument.toTextDocument(TextDocumentOps.scala:201)
	at scala.meta.internal.pc.SemanticdbTextDocumentProvider.textDocument(SemanticdbTextDocumentProvider.scala:54)

Mar 26, 2022 1:26:03 PM scala.meta.internal.pc.CompilerAccess handleError
SEVERE: file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:2: error: ; expected but double constant found
import org.apache.spark3.2.1.SparkContext
                        ^
file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:2: error: ; expected but double constant found
import org.apache.spark3.2.1.SparkContext
                        ^
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.ScalametaParser.syntaxErrorExpected(ScalametaParser.scala:334)
	at scala.meta.internal.parsers.ScalametaParser.accept(ScalametaParser.scala:340)
	at scala.meta.internal.parsers.ScalametaParser.acceptStatSep(ScalametaParser.scala:353)
	at scala.meta.internal.parsers.ScalametaParser.acceptStatSepOpt(ScalametaParser.scala:357)
	at scala.meta.internal.parsers.ScalametaParser.statSeq(ScalametaParser.scala:4157)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$batchSource$1(ScalametaParser.scala:4391)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.batchSource(ScalametaParser.scala:4350)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$source$1(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.source(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.entrypointSource(ScalametaParser.scala:4348)
	at scala.meta.internal.parsers.ScalametaParser.parseSourceImpl(ScalametaParser.scala:132)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$parseSource$1(ScalametaParser.scala:129)
	at scala.meta.internal.parsers.ScalametaParser.parseRuleAfterBOF(ScalametaParser.scala:56)
	at scala.meta.internal.parsers.ScalametaParser.parseRule(ScalametaParser.scala:51)
	at scala.meta.internal.parsers.ScalametaParser.parseSource(ScalametaParser.scala:129)
	at scala.meta.parsers.Parse$.$anonfun$parseSource$1(Parse.scala:29)
	at scala.meta.parsers.Parse$$anon$1.apply(Parse.scala:36)
	at scala.meta.parsers.Api$XtensionParseDialectInput.parse(Api.scala:25)
	at scala.meta.internal.semanticdb.scalac.ParseOps$XtensionCompilationUnitSource.toSource(ParseOps.scala:17)
	at scala.meta.internal.semanticdb.scalac.TextDocumentOps$XtensionCompilationUnitDocument.toTextDocument(TextDocumentOps.scala:201)
	at scala.meta.internal.pc.SemanticdbTextDocumentProvider.textDocument(SemanticdbTextDocumentProvider.scala:54)

Mar 26, 2022 1:26:04 PM scala.meta.internal.pc.CompilerAccess handleError
SEVERE: file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:2: error: ; expected but double constant found
import org.apache.spark3.2.1.SparkContext
                        ^
file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:2: error: ; expected but double constant found
import org.apache.spark3.2.1.SparkContext
                        ^
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.ScalametaParser.syntaxErrorExpected(ScalametaParser.scala:334)
	at scala.meta.internal.parsers.ScalametaParser.accept(ScalametaParser.scala:340)
	at scala.meta.internal.parsers.ScalametaParser.acceptStatSep(ScalametaParser.scala:353)
	at scala.meta.internal.parsers.ScalametaParser.acceptStatSepOpt(ScalametaParser.scala:357)
	at scala.meta.internal.parsers.ScalametaParser.statSeq(ScalametaParser.scala:4157)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$batchSource$1(ScalametaParser.scala:4391)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.batchSource(ScalametaParser.scala:4350)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$source$1(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.source(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.entrypointSource(ScalametaParser.scala:4348)
	at scala.meta.internal.parsers.ScalametaParser.parseSourceImpl(ScalametaParser.scala:132)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$parseSource$1(ScalametaParser.scala:129)
	at scala.meta.internal.parsers.ScalametaParser.parseRuleAfterBOF(ScalametaParser.scala:56)
	at scala.meta.internal.parsers.ScalametaParser.parseRule(ScalametaParser.scala:51)
	at scala.meta.internal.parsers.ScalametaParser.parseSource(ScalametaParser.scala:129)
	at scala.meta.parsers.Parse$.$anonfun$parseSource$1(Parse.scala:29)
	at scala.meta.parsers.Parse$$anon$1.apply(Parse.scala:36)
	at scala.meta.parsers.Api$XtensionParseDialectInput.parse(Api.scala:25)
	at scala.meta.internal.semanticdb.scalac.ParseOps$XtensionCompilationUnitSource.toSource(ParseOps.scala:17)
	at scala.meta.internal.semanticdb.scalac.TextDocumentOps$XtensionCompilationUnitDocument.toTextDocument(TextDocumentOps.scala:201)
	at scala.meta.internal.pc.SemanticdbTextDocumentProvider.textDocument(SemanticdbTextDocumentProvider.scala:54)

Mar 26, 2022 1:26:15 PM scala.meta.internal.pc.CompilerAccess handleError
SEVERE: file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:2: error: ; expected but double constant found
import org.apache.spark3.2.1.SparkContext
                        ^
file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:2: error: ; expected but double constant found
import org.apache.spark3.2.1.SparkContext
                        ^
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.ScalametaParser.syntaxErrorExpected(ScalametaParser.scala:334)
	at scala.meta.internal.parsers.ScalametaParser.accept(ScalametaParser.scala:340)
	at scala.meta.internal.parsers.ScalametaParser.acceptStatSep(ScalametaParser.scala:353)
	at scala.meta.internal.parsers.ScalametaParser.acceptStatSepOpt(ScalametaParser.scala:357)
	at scala.meta.internal.parsers.ScalametaParser.statSeq(ScalametaParser.scala:4157)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$batchSource$1(ScalametaParser.scala:4391)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.batchSource(ScalametaParser.scala:4350)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$source$1(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.source(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.entrypointSource(ScalametaParser.scala:4348)
	at scala.meta.internal.parsers.ScalametaParser.parseSourceImpl(ScalametaParser.scala:132)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$parseSource$1(ScalametaParser.scala:129)
	at scala.meta.internal.parsers.ScalametaParser.parseRuleAfterBOF(ScalametaParser.scala:56)
	at scala.meta.internal.parsers.ScalametaParser.parseRule(ScalametaParser.scala:51)
	at scala.meta.internal.parsers.ScalametaParser.parseSource(ScalametaParser.scala:129)
	at scala.meta.parsers.Parse$.$anonfun$parseSource$1(Parse.scala:29)
	at scala.meta.parsers.Parse$$anon$1.apply(Parse.scala:36)
	at scala.meta.parsers.Api$XtensionParseDialectInput.parse(Api.scala:25)
	at scala.meta.internal.semanticdb.scalac.ParseOps$XtensionCompilationUnitSource.toSource(ParseOps.scala:17)
	at scala.meta.internal.semanticdb.scalac.TextDocumentOps$XtensionCompilationUnitDocument.toTextDocument(TextDocumentOps.scala:201)
	at scala.meta.internal.pc.SemanticdbTextDocumentProvider.textDocument(SemanticdbTextDocumentProvider.scala:54)

Mar 26, 2022 1:26:18 PM scala.meta.internal.pc.CompilerAccess handleError
SEVERE: file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:2: error: ; expected but double constant found
import org.apache.spark3.2.1.SparkContext
                        ^
file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:2: error: ; expected but double constant found
import org.apache.spark3.2.1.SparkContext
                        ^
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.ScalametaParser.syntaxErrorExpected(ScalametaParser.scala:334)
	at scala.meta.internal.parsers.ScalametaParser.accept(ScalametaParser.scala:340)
	at scala.meta.internal.parsers.ScalametaParser.acceptStatSep(ScalametaParser.scala:353)
	at scala.meta.internal.parsers.ScalametaParser.acceptStatSepOpt(ScalametaParser.scala:357)
	at scala.meta.internal.parsers.ScalametaParser.statSeq(ScalametaParser.scala:4157)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$batchSource$1(ScalametaParser.scala:4391)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.batchSource(ScalametaParser.scala:4350)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$source$1(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.source(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.entrypointSource(ScalametaParser.scala:4348)
	at scala.meta.internal.parsers.ScalametaParser.parseSourceImpl(ScalametaParser.scala:132)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$parseSource$1(ScalametaParser.scala:129)
	at scala.meta.internal.parsers.ScalametaParser.parseRuleAfterBOF(ScalametaParser.scala:56)
	at scala.meta.internal.parsers.ScalametaParser.parseRule(ScalametaParser.scala:51)
	at scala.meta.internal.parsers.ScalametaParser.parseSource(ScalametaParser.scala:129)
	at scala.meta.parsers.Parse$.$anonfun$parseSource$1(Parse.scala:29)
	at scala.meta.parsers.Parse$$anon$1.apply(Parse.scala:36)
	at scala.meta.parsers.Api$XtensionParseDialectInput.parse(Api.scala:25)
	at scala.meta.internal.semanticdb.scalac.ParseOps$XtensionCompilationUnitSource.toSource(ParseOps.scala:17)
	at scala.meta.internal.semanticdb.scalac.TextDocumentOps$XtensionCompilationUnitDocument.toTextDocument(TextDocumentOps.scala:201)
	at scala.meta.internal.pc.SemanticdbTextDocumentProvider.textDocument(SemanticdbTextDocumentProvider.scala:54)

Mar 26, 2022 1:26:19 PM scala.meta.internal.pc.CompilerAccess handleError
SEVERE: file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:2: error: ; expected but double constant found
import org.apache.spark3.2.1.SparkContext
                        ^
file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:2: error: ; expected but double constant found
import org.apache.spark3.2.1.SparkContext
                        ^
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.ScalametaParser.syntaxErrorExpected(ScalametaParser.scala:334)
	at scala.meta.internal.parsers.ScalametaParser.accept(ScalametaParser.scala:340)
	at scala.meta.internal.parsers.ScalametaParser.acceptStatSep(ScalametaParser.scala:353)
	at scala.meta.internal.parsers.ScalametaParser.acceptStatSepOpt(ScalametaParser.scala:357)
	at scala.meta.internal.parsers.ScalametaParser.statSeq(ScalametaParser.scala:4157)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$batchSource$1(ScalametaParser.scala:4391)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.batchSource(ScalametaParser.scala:4350)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$source$1(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.source(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.entrypointSource(ScalametaParser.scala:4348)
	at scala.meta.internal.parsers.ScalametaParser.parseSourceImpl(ScalametaParser.scala:132)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$parseSource$1(ScalametaParser.scala:129)
	at scala.meta.internal.parsers.ScalametaParser.parseRuleAfterBOF(ScalametaParser.scala:56)
	at scala.meta.internal.parsers.ScalametaParser.parseRule(ScalametaParser.scala:51)
	at scala.meta.internal.parsers.ScalametaParser.parseSource(ScalametaParser.scala:129)
	at scala.meta.parsers.Parse$.$anonfun$parseSource$1(Parse.scala:29)
	at scala.meta.parsers.Parse$$anon$1.apply(Parse.scala:36)
	at scala.meta.parsers.Api$XtensionParseDialectInput.parse(Api.scala:25)
	at scala.meta.internal.semanticdb.scalac.ParseOps$XtensionCompilationUnitSource.toSource(ParseOps.scala:17)
	at scala.meta.internal.semanticdb.scalac.TextDocumentOps$XtensionCompilationUnitDocument.toTextDocument(TextDocumentOps.scala:201)
	at scala.meta.internal.pc.SemanticdbTextDocumentProvider.textDocument(SemanticdbTextDocumentProvider.scala:54)

Mar 26, 2022 1:26:19 PM scala.meta.internal.pc.CompilerAccess handleError
SEVERE: file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:2: error: ; expected but double constant found
import org.apache.spark3.2.1.SparkContext
                        ^
file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:2: error: ; expected but double constant found
import org.apache.spark3.2.1.SparkContext
                        ^
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.ScalametaParser.syntaxErrorExpected(ScalametaParser.scala:334)
	at scala.meta.internal.parsers.ScalametaParser.accept(ScalametaParser.scala:340)
	at scala.meta.internal.parsers.ScalametaParser.acceptStatSep(ScalametaParser.scala:353)
	at scala.meta.internal.parsers.ScalametaParser.acceptStatSepOpt(ScalametaParser.scala:357)
	at scala.meta.internal.parsers.ScalametaParser.statSeq(ScalametaParser.scala:4157)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$batchSource$1(ScalametaParser.scala:4391)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.batchSource(ScalametaParser.scala:4350)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$source$1(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.source(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.entrypointSource(ScalametaParser.scala:4348)
	at scala.meta.internal.parsers.ScalametaParser.parseSourceImpl(ScalametaParser.scala:132)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$parseSource$1(ScalametaParser.scala:129)
	at scala.meta.internal.parsers.ScalametaParser.parseRuleAfterBOF(ScalametaParser.scala:56)
	at scala.meta.internal.parsers.ScalametaParser.parseRule(ScalametaParser.scala:51)
	at scala.meta.internal.parsers.ScalametaParser.parseSource(ScalametaParser.scala:129)
	at scala.meta.parsers.Parse$.$anonfun$parseSource$1(Parse.scala:29)
	at scala.meta.parsers.Parse$$anon$1.apply(Parse.scala:36)
	at scala.meta.parsers.Api$XtensionParseDialectInput.parse(Api.scala:25)
	at scala.meta.internal.semanticdb.scalac.ParseOps$XtensionCompilationUnitSource.toSource(ParseOps.scala:17)
	at scala.meta.internal.semanticdb.scalac.TextDocumentOps$XtensionCompilationUnitDocument.toTextDocument(TextDocumentOps.scala:201)
	at scala.meta.internal.pc.SemanticdbTextDocumentProvider.textDocument(SemanticdbTextDocumentProvider.scala:54)

Mar 26, 2022 1:26:20 PM scala.meta.internal.pc.CompilerAccess handleError
SEVERE: file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:2: error: ; expected but double constant found
import org.apache.spark3.2.1.SparkContext
                        ^
file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:2: error: ; expected but double constant found
import org.apache.spark3.2.1.SparkContext
                        ^
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.ScalametaParser.syntaxErrorExpected(ScalametaParser.scala:334)
	at scala.meta.internal.parsers.ScalametaParser.accept(ScalametaParser.scala:340)
	at scala.meta.internal.parsers.ScalametaParser.acceptStatSep(ScalametaParser.scala:353)
	at scala.meta.internal.parsers.ScalametaParser.acceptStatSepOpt(ScalametaParser.scala:357)
	at scala.meta.internal.parsers.ScalametaParser.statSeq(ScalametaParser.scala:4157)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$batchSource$1(ScalametaParser.scala:4391)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.batchSource(ScalametaParser.scala:4350)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$source$1(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.source(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.entrypointSource(ScalametaParser.scala:4348)
	at scala.meta.internal.parsers.ScalametaParser.parseSourceImpl(ScalametaParser.scala:132)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$parseSource$1(ScalametaParser.scala:129)
	at scala.meta.internal.parsers.ScalametaParser.parseRuleAfterBOF(ScalametaParser.scala:56)
	at scala.meta.internal.parsers.ScalametaParser.parseRule(ScalametaParser.scala:51)
	at scala.meta.internal.parsers.ScalametaParser.parseSource(ScalametaParser.scala:129)
	at scala.meta.parsers.Parse$.$anonfun$parseSource$1(Parse.scala:29)
	at scala.meta.parsers.Parse$$anon$1.apply(Parse.scala:36)
	at scala.meta.parsers.Api$XtensionParseDialectInput.parse(Api.scala:25)
	at scala.meta.internal.semanticdb.scalac.ParseOps$XtensionCompilationUnitSource.toSource(ParseOps.scala:17)
	at scala.meta.internal.semanticdb.scalac.TextDocumentOps$XtensionCompilationUnitDocument.toTextDocument(TextDocumentOps.scala:201)
	at scala.meta.internal.pc.SemanticdbTextDocumentProvider.textDocument(SemanticdbTextDocumentProvider.scala:54)

Mar 26, 2022 1:26:21 PM scala.meta.internal.pc.CompilerAccess handleError
SEVERE: file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:2: error: ; expected but double constant found
import org.apache.spark3.2.1.SparkContext
                        ^
file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:2: error: ; expected but double constant found
import org.apache.spark3.2.1.SparkContext
                        ^
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.ScalametaParser.syntaxErrorExpected(ScalametaParser.scala:334)
	at scala.meta.internal.parsers.ScalametaParser.accept(ScalametaParser.scala:340)
	at scala.meta.internal.parsers.ScalametaParser.acceptStatSep(ScalametaParser.scala:353)
	at scala.meta.internal.parsers.ScalametaParser.acceptStatSepOpt(ScalametaParser.scala:357)
	at scala.meta.internal.parsers.ScalametaParser.statSeq(ScalametaParser.scala:4157)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$batchSource$1(ScalametaParser.scala:4391)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.batchSource(ScalametaParser.scala:4350)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$source$1(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.source(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.entrypointSource(ScalametaParser.scala:4348)
	at scala.meta.internal.parsers.ScalametaParser.parseSourceImpl(ScalametaParser.scala:132)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$parseSource$1(ScalametaParser.scala:129)
	at scala.meta.internal.parsers.ScalametaParser.parseRuleAfterBOF(ScalametaParser.scala:56)
	at scala.meta.internal.parsers.ScalametaParser.parseRule(ScalametaParser.scala:51)
	at scala.meta.internal.parsers.ScalametaParser.parseSource(ScalametaParser.scala:129)
	at scala.meta.parsers.Parse$.$anonfun$parseSource$1(Parse.scala:29)
	at scala.meta.parsers.Parse$$anon$1.apply(Parse.scala:36)
	at scala.meta.parsers.Api$XtensionParseDialectInput.parse(Api.scala:25)
	at scala.meta.internal.semanticdb.scalac.ParseOps$XtensionCompilationUnitSource.toSource(ParseOps.scala:17)
	at scala.meta.internal.semanticdb.scalac.TextDocumentOps$XtensionCompilationUnitDocument.toTextDocument(TextDocumentOps.scala:201)
	at scala.meta.internal.pc.SemanticdbTextDocumentProvider.textDocument(SemanticdbTextDocumentProvider.scala:54)

Mar 26, 2022 1:26:22 PM scala.meta.internal.pc.CompilerAccess handleError
SEVERE: file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:2: error: ; expected but double constant found
import org.apache.spark3.2.1.SparkContext
                        ^
file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:2: error: ; expected but double constant found
import org.apache.spark3.2.1.SparkContext
                        ^
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.ScalametaParser.syntaxErrorExpected(ScalametaParser.scala:334)
	at scala.meta.internal.parsers.ScalametaParser.accept(ScalametaParser.scala:340)
	at scala.meta.internal.parsers.ScalametaParser.acceptStatSep(ScalametaParser.scala:353)
	at scala.meta.internal.parsers.ScalametaParser.acceptStatSepOpt(ScalametaParser.scala:357)
	at scala.meta.internal.parsers.ScalametaParser.statSeq(ScalametaParser.scala:4157)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$batchSource$1(ScalametaParser.scala:4391)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.batchSource(ScalametaParser.scala:4350)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$source$1(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.source(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.entrypointSource(ScalametaParser.scala:4348)
	at scala.meta.internal.parsers.ScalametaParser.parseSourceImpl(ScalametaParser.scala:132)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$parseSource$1(ScalametaParser.scala:129)
	at scala.meta.internal.parsers.ScalametaParser.parseRuleAfterBOF(ScalametaParser.scala:56)
	at scala.meta.internal.parsers.ScalametaParser.parseRule(ScalametaParser.scala:51)
	at scala.meta.internal.parsers.ScalametaParser.parseSource(ScalametaParser.scala:129)
	at scala.meta.parsers.Parse$.$anonfun$parseSource$1(Parse.scala:29)
	at scala.meta.parsers.Parse$$anon$1.apply(Parse.scala:36)
	at scala.meta.parsers.Api$XtensionParseDialectInput.parse(Api.scala:25)
	at scala.meta.internal.semanticdb.scalac.ParseOps$XtensionCompilationUnitSource.toSource(ParseOps.scala:17)
	at scala.meta.internal.semanticdb.scalac.TextDocumentOps$XtensionCompilationUnitDocument.toTextDocument(TextDocumentOps.scala:201)
	at scala.meta.internal.pc.SemanticdbTextDocumentProvider.textDocument(SemanticdbTextDocumentProvider.scala:54)

Mar 26, 2022 1:26:22 PM scala.meta.internal.pc.CompilerAccess handleError
SEVERE: file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:2: error: ; expected but double constant found
import org.apache.spark3.2.1.SparkContext
                        ^
file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:2: error: ; expected but double constant found
import org.apache.spark3.2.1.SparkContext
                        ^
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.ScalametaParser.syntaxErrorExpected(ScalametaParser.scala:334)
	at scala.meta.internal.parsers.ScalametaParser.accept(ScalametaParser.scala:340)
	at scala.meta.internal.parsers.ScalametaParser.acceptStatSep(ScalametaParser.scala:353)
	at scala.meta.internal.parsers.ScalametaParser.acceptStatSepOpt(ScalametaParser.scala:357)
	at scala.meta.internal.parsers.ScalametaParser.statSeq(ScalametaParser.scala:4157)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$batchSource$1(ScalametaParser.scala:4391)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.batchSource(ScalametaParser.scala:4350)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$source$1(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.source(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.entrypointSource(ScalametaParser.scala:4348)
	at scala.meta.internal.parsers.ScalametaParser.parseSourceImpl(ScalametaParser.scala:132)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$parseSource$1(ScalametaParser.scala:129)
	at scala.meta.internal.parsers.ScalametaParser.parseRuleAfterBOF(ScalametaParser.scala:56)
	at scala.meta.internal.parsers.ScalametaParser.parseRule(ScalametaParser.scala:51)
	at scala.meta.internal.parsers.ScalametaParser.parseSource(ScalametaParser.scala:129)
	at scala.meta.parsers.Parse$.$anonfun$parseSource$1(Parse.scala:29)
	at scala.meta.parsers.Parse$$anon$1.apply(Parse.scala:36)
	at scala.meta.parsers.Api$XtensionParseDialectInput.parse(Api.scala:25)
	at scala.meta.internal.semanticdb.scalac.ParseOps$XtensionCompilationUnitSource.toSource(ParseOps.scala:17)
	at scala.meta.internal.semanticdb.scalac.TextDocumentOps$XtensionCompilationUnitDocument.toTextDocument(TextDocumentOps.scala:201)
	at scala.meta.internal.pc.SemanticdbTextDocumentProvider.textDocument(SemanticdbTextDocumentProvider.scala:54)

2022.03.26 13:26:22 INFO  time: code lens generation in 1.1s
Mar 26, 2022 1:26:22 PM scala.meta.internal.pc.CompilerAccess handleError
SEVERE: file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:2: error: ; expected but double constant found
import org.apache.spark3.2.1.SparkContext
                        ^
file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:2: error: ; expected but double constant found
import org.apache.spark3.2.1.SparkContext
                        ^
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.ScalametaParser.syntaxErrorExpected(ScalametaParser.scala:334)
	at scala.meta.internal.parsers.ScalametaParser.accept(ScalametaParser.scala:340)
	at scala.meta.internal.parsers.ScalametaParser.acceptStatSep(ScalametaParser.scala:353)
	at scala.meta.internal.parsers.ScalametaParser.acceptStatSepOpt(ScalametaParser.scala:357)
	at scala.meta.internal.parsers.ScalametaParser.statSeq(ScalametaParser.scala:4157)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$batchSource$1(ScalametaParser.scala:4391)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.batchSource(ScalametaParser.scala:4350)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$source$1(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.source(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.entrypointSource(ScalametaParser.scala:4348)
	at scala.meta.internal.parsers.ScalametaParser.parseSourceImpl(ScalametaParser.scala:132)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$parseSource$1(ScalametaParser.scala:129)
	at scala.meta.internal.parsers.ScalametaParser.parseRuleAfterBOF(ScalametaParser.scala:56)
	at scala.meta.internal.parsers.ScalametaParser.parseRule(ScalametaParser.scala:51)
	at scala.meta.internal.parsers.ScalametaParser.parseSource(ScalametaParser.scala:129)
	at scala.meta.parsers.Parse$.$anonfun$parseSource$1(Parse.scala:29)
	at scala.meta.parsers.Parse$$anon$1.apply(Parse.scala:36)
	at scala.meta.parsers.Api$XtensionParseDialectInput.parse(Api.scala:25)
	at scala.meta.internal.semanticdb.scalac.ParseOps$XtensionCompilationUnitSource.toSource(ParseOps.scala:17)
	at scala.meta.internal.semanticdb.scalac.TextDocumentOps$XtensionCompilationUnitDocument.toTextDocument(TextDocumentOps.scala:201)
	at scala.meta.internal.pc.SemanticdbTextDocumentProvider.textDocument(SemanticdbTextDocumentProvider.scala:54)

Mar 26, 2022 1:26:23 PM scala.meta.internal.pc.CompilerAccess handleError
SEVERE: file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:2: error: ; expected but double constant found
import org.apache.spark3.2.1.SparkContext
                        ^
file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:2: error: ; expected but double constant found
import org.apache.spark3.2.1.SparkContext
                        ^
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.ScalametaParser.syntaxErrorExpected(ScalametaParser.scala:334)
	at scala.meta.internal.parsers.ScalametaParser.accept(ScalametaParser.scala:340)
	at scala.meta.internal.parsers.ScalametaParser.acceptStatSep(ScalametaParser.scala:353)
	at scala.meta.internal.parsers.ScalametaParser.acceptStatSepOpt(ScalametaParser.scala:357)
	at scala.meta.internal.parsers.ScalametaParser.statSeq(ScalametaParser.scala:4157)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$batchSource$1(ScalametaParser.scala:4391)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.batchSource(ScalametaParser.scala:4350)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$source$1(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.source(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.entrypointSource(ScalametaParser.scala:4348)
	at scala.meta.internal.parsers.ScalametaParser.parseSourceImpl(ScalametaParser.scala:132)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$parseSource$1(ScalametaParser.scala:129)
	at scala.meta.internal.parsers.ScalametaParser.parseRuleAfterBOF(ScalametaParser.scala:56)
	at scala.meta.internal.parsers.ScalametaParser.parseRule(ScalametaParser.scala:51)
	at scala.meta.internal.parsers.ScalametaParser.parseSource(ScalametaParser.scala:129)
	at scala.meta.parsers.Parse$.$anonfun$parseSource$1(Parse.scala:29)
	at scala.meta.parsers.Parse$$anon$1.apply(Parse.scala:36)
	at scala.meta.parsers.Api$XtensionParseDialectInput.parse(Api.scala:25)
	at scala.meta.internal.semanticdb.scalac.ParseOps$XtensionCompilationUnitSource.toSource(ParseOps.scala:17)
	at scala.meta.internal.semanticdb.scalac.TextDocumentOps$XtensionCompilationUnitDocument.toTextDocument(TextDocumentOps.scala:201)
	at scala.meta.internal.pc.SemanticdbTextDocumentProvider.textDocument(SemanticdbTextDocumentProvider.scala:54)

Mar 26, 2022 1:26:23 PM scala.meta.internal.pc.CompilerAccess handleError
SEVERE: file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:2: error: ; expected but identifier found
import org.apache.spark-3.2.1.SparkContext
                       ^
file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:2: error: ; expected but identifier found
import org.apache.spark-3.2.1.SparkContext
                       ^
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.ScalametaParser.syntaxErrorExpected(ScalametaParser.scala:334)
	at scala.meta.internal.parsers.ScalametaParser.accept(ScalametaParser.scala:340)
	at scala.meta.internal.parsers.ScalametaParser.acceptStatSep(ScalametaParser.scala:353)
	at scala.meta.internal.parsers.ScalametaParser.acceptStatSepOpt(ScalametaParser.scala:357)
	at scala.meta.internal.parsers.ScalametaParser.statSeq(ScalametaParser.scala:4157)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$batchSource$1(ScalametaParser.scala:4391)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.batchSource(ScalametaParser.scala:4350)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$source$1(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.source(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.entrypointSource(ScalametaParser.scala:4348)
	at scala.meta.internal.parsers.ScalametaParser.parseSourceImpl(ScalametaParser.scala:132)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$parseSource$1(ScalametaParser.scala:129)
	at scala.meta.internal.parsers.ScalametaParser.parseRuleAfterBOF(ScalametaParser.scala:56)
	at scala.meta.internal.parsers.ScalametaParser.parseRule(ScalametaParser.scala:51)
	at scala.meta.internal.parsers.ScalametaParser.parseSource(ScalametaParser.scala:129)
	at scala.meta.parsers.Parse$.$anonfun$parseSource$1(Parse.scala:29)
	at scala.meta.parsers.Parse$$anon$1.apply(Parse.scala:36)
	at scala.meta.parsers.Api$XtensionParseDialectInput.parse(Api.scala:25)
	at scala.meta.internal.semanticdb.scalac.ParseOps$XtensionCompilationUnitSource.toSource(ParseOps.scala:17)
	at scala.meta.internal.semanticdb.scalac.TextDocumentOps$XtensionCompilationUnitDocument.toTextDocument(TextDocumentOps.scala:201)
	at scala.meta.internal.pc.SemanticdbTextDocumentProvider.textDocument(SemanticdbTextDocumentProvider.scala:54)

2022.03.26 13:26:23 WARN  no build target for: /Users/Tarin/Documents/Projects/Programming/flight-data/src/main/scala/FlightData.scala
Mar 26, 2022 1:26:24 PM scala.meta.internal.pc.CompilerAccess handleError
SEVERE: file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:2: error: ; expected but identifier found
import org.apache.spark-3.2.1.SparkContext
                       ^
file%3A%2F%2F%2FUsers%2FTarin%2FDocuments%2FProjects%2FProgramming%2Fflight-data%2Fsrc%2Fmain%2Fscala%2FFlightData.scala:2: error: ; expected but identifier found
import org.apache.spark-3.2.1.SparkContext
                       ^
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:16)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:17)
	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
	at scala.meta.internal.parsers.ScalametaParser.syntaxErrorExpected(ScalametaParser.scala:334)
	at scala.meta.internal.parsers.ScalametaParser.accept(ScalametaParser.scala:340)
	at scala.meta.internal.parsers.ScalametaParser.acceptStatSep(ScalametaParser.scala:353)
	at scala.meta.internal.parsers.ScalametaParser.acceptStatSepOpt(ScalametaParser.scala:357)
	at scala.meta.internal.parsers.ScalametaParser.statSeq(ScalametaParser.scala:4157)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$batchSource$1(ScalametaParser.scala:4391)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.batchSource(ScalametaParser.scala:4350)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$source$1(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:270)
	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:311)
	at scala.meta.internal.parsers.ScalametaParser.source(ScalametaParser.scala:4343)
	at scala.meta.internal.parsers.ScalametaParser.entrypointSource(ScalametaParser.scala:4348)
	at scala.meta.internal.parsers.ScalametaParser.parseSourceImpl(ScalametaParser.scala:132)
	at scala.meta.internal.parsers.ScalametaParser.$anonfun$parseSource$1(ScalametaParser.scala:129)
	at scala.meta.internal.parsers.ScalametaParser.parseRuleAfterBOF(ScalametaParser.scala:56)
	at scala.meta.internal.parsers.ScalametaParser.parseRule(ScalametaParser.scala:51)
	at scala.meta.internal.parsers.ScalametaParser.parseSource(ScalametaParser.scala:129)
	at scala.meta.parsers.Parse$.$anonfun$parseSource$1(Parse.scala:29)
	at scala.meta.parsers.Parse$$anon$1.apply(Parse.scala:36)
	at scala.meta.parsers.Api$XtensionParseDialectInput.parse(Api.scala:25)
	at scala.meta.internal.semanticdb.scalac.ParseOps$XtensionCompilationUnitSource.toSource(ParseOps.scala:17)
	at scala.meta.internal.semanticdb.scalac.TextDocumentOps$XtensionCompilationUnitDocument.toTextDocument(TextDocumentOps.scala:201)
	at scala.meta.internal.pc.SemanticdbTextDocumentProvider.textDocument(SemanticdbTextDocumentProvider.scala:54)

